<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content='The `pull_data_synapse` function accesses the specified
 version of the clinical and genomic GENIE BPC data from
 Synapse
 and reads it into the R environment.
  Documentation corresponding to the clinical data files
  can also be found on Synapse in the Analytic Data Guide:
NSCLC v1.1-Consortium Analytic Data Guide
NSCLC v2.1-Consortium Analytic Data Guide
NSCLC v2.0-Public Analytic Data Guide
CRC v1.1-Consortium Analytic Data Guide
CRC v1.2-Consortium Analytic Data Guide
BrCa v1.1-Consortium Analytic Data Guide
Users must log in to Synapse to access the data successfully.
To set your Synapse credentials during each session, call:
`set_synapse_credentials(username = "your_username", password = "your_password")`
To store authentication information in your environmental variables, add the
following to your .Renviron file (tip: you can use usethis::edit_r_environ() to easily open/edit this file):
`SYNAPSE_USERNAME = &amp;lt;your-username&amp;gt;`
`SYNAPSE_PASSWORD = &amp;lt;your-password&amp;gt;`
Alternatively, you can pass your username and password to each individual data pull function if preferred,
although it is recommended that you manage your passwords outside of your scripts for security purposes.'><title>Obtain clinical &amp; genomic data files for GENIE BPC Project — pull_data_synapse • genieBPC</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.1.0/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.1.0/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Obtain clinical &amp; genomic data files for GENIE BPC Project — pull_data_synapse"><meta property="og:description" content='The `pull_data_synapse` function accesses the specified
 version of the clinical and genomic GENIE BPC data from
 Synapse
 and reads it into the R environment.
  Documentation corresponding to the clinical data files
  can also be found on Synapse in the Analytic Data Guide:
NSCLC v1.1-Consortium Analytic Data Guide
NSCLC v2.1-Consortium Analytic Data Guide
NSCLC v2.0-Public Analytic Data Guide
CRC v1.1-Consortium Analytic Data Guide
CRC v1.2-Consortium Analytic Data Guide
BrCa v1.1-Consortium Analytic Data Guide
Users must log in to Synapse to access the data successfully.
To set your Synapse credentials during each session, call:
`set_synapse_credentials(username = "your_username", password = "your_password")`
To store authentication information in your environmental variables, add the
following to your .Renviron file (tip: you can use usethis::edit_r_environ() to easily open/edit this file):
`SYNAPSE_USERNAME = &amp;lt;your-username&amp;gt;`
`SYNAPSE_PASSWORD = &amp;lt;your-password&amp;gt;`
Alternatively, you can pass your username and password to each individual data pull function if preferred,
although it is recommended that you manage your passwords outside of your scripts for security purposes.'><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-primary"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">genieBPC</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.1.0.9002</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/clinical_data_structure_vignette.html">Tutorial: GENIE BPC Data Structure</a>
    <a class="dropdown-item" href="../articles/create_analytic_cohort_vignette.html">Tutorial: create_analytic_cohort</a>
    <a class="dropdown-item" href="../articles/drug_regimen_sunburst_vignette.html">Tutorial: drug_regimen_sunburst</a>
    <a class="dropdown-item" href="../articles/pull_data_synapse_vignette.html">Tutorial: pull_data_synapse</a>
    <a class="dropdown-item" href="../articles/select_unique_ngs_vignette.html">Tutorial: select_unique_ngs</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">Changelog</a>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/GENIE-BPC/genieBPC/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Obtain clinical &amp; genomic data files for GENIE BPC Project</h1>
      <small class="dont-index">Source: <a href="https://github.com/GENIE-BPC/genieBPC/blob/HEAD/R/pull_data_synapse.R" class="external-link"><code>R/pull_data_synapse.R</code></a></small>
      <div class="d-none name"><code>pull_data_synapse.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>The `pull_data_synapse` function accesses the specified
 version of the clinical and genomic GENIE BPC data from
 <a href="https://www.synapse.org/#!Synapse:syn21226493/wiki/599164" class="external-link">Synapse</a>
 and reads it into the R environment.
  Documentation corresponding to the clinical data files
  can also be found on Synapse in the Analytic Data Guide:</p><ul><li><p><a href="https://www.synapse.org/#!Synapse:syn23002641" class="external-link">NSCLC v1.1-Consortium Analytic Data Guide</a></p></li>
<li><p><a href="https://www.synapse.org/#!Synapse:syn26008058" class="external-link">NSCLC v2.1-Consortium Analytic Data Guide</a></p></li>
<li><p><a href="https://www.synapse.org/#!Synapse:syn30557304" class="external-link">NSCLC v2.0-Public Analytic Data Guide</a></p></li>
<li><p><a href="https://www.synapse.org/#!Synapse:syn23764204" class="external-link">CRC v1.1-Consortium Analytic Data Guide</a></p></li>
<li><p><a href="https://www.synapse.org/#!Synapse:syn26077308" class="external-link">CRC v1.2-Consortium Analytic Data Guide</a></p></li>
<li><p><a href="https://www.synapse.org/#!Synapse:syn26077313" class="external-link">BrCa v1.1-Consortium Analytic Data Guide</a></p></li>
</ul><p>Users must log in to Synapse to access the data successfully.
To set your Synapse credentials during each session, call:
`set_synapse_credentials(username = "your_username", password = "your_password")`
To store authentication information in your environmental variables, add the
following to your .Renviron file (tip: you can use usethis::edit_r_environ() to easily open/edit this file):
`SYNAPSE_USERNAME = &lt;your-username&gt;`
`SYNAPSE_PASSWORD = &lt;your-password&gt;`
Alternatively, you can pass your username and password to each individual data pull function if preferred,
although it is recommended that you manage your passwords outside of your scripts for security purposes.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">pull_data_synapse</span><span class="op">(</span></span>
<span>  cohort <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  version <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  download_location <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  username <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  password <span class="op">=</span> <span class="cn">NULL</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>cohort</dt>
<dd><p>Vector or list specifying the cohort(s) of interest.
Must be one of "NSCLC" (Non-Small Cell Lung Cancer),
 "CRC" (Colorectal Cancer), or "BrCa" (Breast Cancer).</p></dd>


<dt>version</dt>
<dd><p>Vector specifying the version of the data. Must be one of the following:
"v1.1-consortium", "v1.2-consortium", "v2.1-consortium", "v2.0-public". When entering multiple cohorts, the order of the
version numbers corresponds to the order that the cohorts
are specified; the cohort and version number must be in
the same order in order to pull the correct data. See examples below.</p></dd>


<dt>download_location</dt>
<dd><p>if `NULL` (default), data will be returned as a list of dataframes with
requested data as list items. Otherwise, specify a folder path to have data automatically downloaded there.
When a path is specified, data are not read into the R environment.</p></dd>


<dt>username</dt>
<dd><p>Synapse username</p></dd>


<dt>password</dt>
<dd><p>Synapse password</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p>Returns a nested list of clinical and genomic data corresponding to
the specified cohort(s).</p>
    </div>
    <div class="section level2">
    <h2 id="details">Details<a class="anchor" aria-label="anchor" href="#details"></a></h2>
    <p>See the <a href="https://genie-bpc.github.io/genieBPC/articles/pull_data_synapse_vignette.html">pull_data_synapse vignette</a>
for further documentation and examples.</p>
    </div>
    <div class="section level2">
    <h2 id="author">Author<a class="anchor" aria-label="anchor" href="#author"></a></h2>
    <p>Karissa Whiting, Michael Curry</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co"># \donttest{</span></span></span>
<span class="r-in"><span><span class="kw">if</span><span class="op">(</span><span class="fu">genieBPC</span><span class="fu">::</span><span class="fu"><a href="dot-is_connected_to_genie.html">.is_connected_to_genie</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span> <span class="op">{</span></span></span>
<span class="r-in"><span><span class="co"># Example 1 ----------------------------------</span></span></span>
<span class="r-in"><span><span class="co"># Pull non-small cell lung cancer data</span></span></span>
<span class="r-in"><span><span class="fu"><a href="set_synapse_credentials.html">set_synapse_credentials</a></span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">results</span> <span class="op">&lt;-</span> <span class="fu">pull_data_synapse</span><span class="op">(</span>cohort <span class="op">=</span> <span class="st">"NSCLC"</span>, version <span class="op">=</span> <span class="st">"v2.1-consortium"</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">results</span><span class="op">$</span><span class="va">NSCLC_v2.1</span><span class="op">)</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># Example 2 ----------------------------------</span></span></span>
<span class="r-in"><span><span class="co"># Pull the most recent non-small cell lung cancer</span></span></span>
<span class="r-in"><span><span class="co"># data and the most recent colorectal cancer data</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span> <span class="va">results2</span> <span class="op">&lt;-</span> <span class="fu">pull_data_synapse</span><span class="op">(</span></span></span>
<span class="r-in"><span>   cohort <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"NSCLC"</span>, <span class="st">"CRC"</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>   version <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"v2.1-consortium"</span>, <span class="st">"v1.2-consortium"</span><span class="op">)</span></span></span>
<span class="r-in"><span> <span class="op">)</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">results2</span><span class="op">)</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># Example 3 ----------------------------------</span></span></span>
<span class="r-in"><span><span class="co"># Pull version 2.1 for non-small cell lung cancer</span></span></span>
<span class="r-in"><span><span class="co"># and version 1.1 for colorectal cancer data</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span> <span class="va">results3</span> <span class="op">&lt;-</span> <span class="fu">pull_data_synapse</span><span class="op">(</span></span></span>
<span class="r-in"><span>   cohort <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"CRC"</span>, <span class="st">"BrCa"</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>   version <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"v1.2-consortium"</span>, <span class="st">"v1.1-consortium"</span><span class="op">)</span></span></span>
<span class="r-in"><span> <span class="op">)</span></span></span>
<span class="r-in"><span>  <span class="fu"><a href="https://rdrr.io/r/utils/str.html" class="external-link">str</a></span><span class="op">(</span><span class="va">results3</span><span class="op">)</span></span></span>
<span class="r-in"><span>  <span class="op">}</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> You are now connected to synapse as <span style="color: #00BB00;">bstgeniebpc@mskcc.org</span> for this R session!</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">pt_char</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_index</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_non_index</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_drugs</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_imaging</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_pathology</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_md</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cpt</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">fusions</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cna</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">mutations_extended</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">pt_char</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_index</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_non_index</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_drugs</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_imaging</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_pathology</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_md</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">tumor_marker</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cpt</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">fusions</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cna</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">mutations_extended</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">pt_char</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_index</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_non_index</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_drugs</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_imaging</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_pathology</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_md</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cpt</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">fusions</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cna</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">mutations_extended</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">pt_char</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_index</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_non_index</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_drugs</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_imaging</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_pathology</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_md</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">tumor_marker</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cpt</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">fusions</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cna</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">mutations_extended</span> has been imported for <span style="color: #0000BB;">"BrCa_v1.1"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">pt_char</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_index</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_non_index</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_drugs</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_imaging</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_pathology</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_md</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">tumor_marker</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cpt</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">fusions</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cna</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">mutations_extended</span> has been imported for <span style="color: #0000BB;">"CRC_v1.2"</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> List of 2</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  $ BrCa_v1.1:List of 12</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ pt_char           :'data.frame':	1130 obs. of  40 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                    : chr [1:1130] "BrCa" "BrCa" "BrCa" "BrCa" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id                 : chr [1:1130] "GENIE-DFCI-000183" "GENIE-DFCI-000226" "GENIE-DFCI-000275" "GENIE-DFCI-000284" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution               : chr [1:1130] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ redacted                  : chr [1:1130] "No" "No" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ birth_year                : int [1:1130] 1967 1970 1966 1969 1977 1973 1969 1968 1958 1961 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_ethnicity_code     : chr [1:1130] "Non-Spanish; non-Hispanic" "Non-Spanish; non-Hispanic" "Non-Spanish; non-Hispanic" "Non-Spanish; non-Hispanic" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_race_code_primary  : chr [1:1130] "White" "White" "White" "White" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_race_code_secondary: chr [1:1130] "No further race documented" "No further race documented" "No further race documented" "No further race documented" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_race_code_tertiary : chr [1:1130] "No further race documented" "No further race documented" "No further race documented" "No further race documented" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_sex_code           : chr [1:1130] "Female" "Female" "Female" "Female" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ hybrid_death_int          : int [1:1130] NA NA 18867 NA 15479 16099 NA 17090 20655 20834 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ hybrid_death_source       : chr [1:1130] "" "" "NDI" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ last_oncvisit_int         : int [1:1130] 19485 18498 18830 18622 15471 16092 18622 17033 20619 20824 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ last_alive_int            : int [1:1130] 19485 18498 18839 18622 15471 16099 18635 17044 20653 20824 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ last_anyvisit_int         : int [1:1130] 19485 18498 18830 18622 15471 16092 18622 17033 20629 20824 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ enroll_hospice_int        : int [1:1130] NA NA 18824 NA 15471 16092 NA NA 20629 20811 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_lastalive_int         : int [1:1130] 19485 18498 18839 18622 15471 16099 18635 17044 20653 20824 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_lastalive_int_mos     : num [1:1130] 641 608 620 613 509 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_lastalive_int_yrs     : num [1:1130] 53.3 50.6 51.6 51 42.4 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ age_death_yrs             : int [1:1130] NA NA 51 NA 42 44 NA 46 56 57 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ age_last_fu_yrs           : int [1:1130] 53 50 51 50 42 44 51 46 56 57 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_cancers                 : int [1:1130] 1 1 2 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_cancers_index           : int [1:1130] 1 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_regimens_pt             : int [1:1130] 12 1 8 5 14 8 7 13 8 12 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_imaging_reports_pt      : int [1:1130] 75 4 38 3 71 37 45 157 41 47 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_ct_pt             : int [1:1130] 28 1 16 0 15 24 33 96 22 22 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_mri_pt            : int [1:1130] 6 2 17 1 9 10 9 29 6 3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_pet_ct_pt         : int [1:1130] 18 0 0 0 37 0 1 8 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_mammog_pt         : int [1:1130] 7 0 1 1 5 1 2 0 8 5 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_bone_pt           : int [1:1130] 16 0 4 0 4 2 0 24 4 16 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_other_pt          : int [1:1130] 0 1 0 1 1 0 0 0 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_path_reports_pt         : int [1:1130] 7 5 11 5 10 7 4 11 6 10 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_md_notes_pt             : int [1:1130] 125 10 39 10 98 24 55 30 26 58 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_pt                   : int [1:1130] 121 0 46 0 101 28 0 33 25 74 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_ca15_3_pt            : int [1:1130] 55 0 14 0 37 15 0 0 0 24 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_ca19_9_pt            : int [1:1130] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_ca2729_pt            : int [1:1130] 66 0 32 0 64 13 0 33 25 50 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_cea_pt               : int [1:1130] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_cpt_pt                  : int [1:1130] 1 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version           : num [1:1130] 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ ca_dx_index       :'data.frame':	1141 obs. of  159 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                    : chr [1:1141] "BrCa" "BrCa" "BrCa" "BrCa" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id                 : chr [1:1141] "GENIE-DFCI-000183" "GENIE-DFCI-000226" "GENIE-DFCI-000275" "GENIE-DFCI-000284" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution               : chr [1:1141] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_seq                    : int [1:1141] 0 0 2 0 0 0 0 0 0 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ redcap_ca_index           : chr [1:1141] "Yes" "Yes" "Yes" "Yes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_cadx_int               : int [1:1141] 13201 15959 16786 16995 11342 15286 15471 10310 17941 17560 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_dx_how                 : chr [1:1141] "Pathology" "Pathology" "Pathology" "Pathology" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_next_ca_days          : int [1:1141] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_first_index_ca         : int [1:1141] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_first_contact_int  : int [1:1141] 13209 16074 16786 17097 11353 15451 15605 10310 17967 17610 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_age                    : int [1:1141] 36 43 45 46 31 41 42 28 49 48 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ age_dx                    : int [1:1141] 36 43 45 46 31 41 42 28 49 48 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_site_cd            : chr [1:1141] "C502" "C504" "C508" "C504" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_site                   : chr [1:1141] "C50.8    Overlapping lesion of breast" "C50.8    Overlapping lesion of breast" "C50.9    Breast NOS" "C50.4    Upper-outer quadrant of breast" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_d_site                 : chr [1:1141] "C50.8" "C50.8" "C50.9" "C50.4" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ site_recode               : chr [1:1141] "Breast" "Breast" "Breast" "Breast" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_type                   : chr [1:1141] "Breast Cancer" "Breast Cancer" "Breast Cancer" "Breast Cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_histology_cd       : int [1:1141] 85003 85223 85223 85003 85003 85003 85003 85003 85203 85073 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_hist_adeno_squamous    : chr [1:1141] "Carcinoma" "Carcinoma" "Carcinoma" "Carcinoma" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_hist_brca              : chr [1:1141] "Invasive ductal carcinoma" "Invasive ductal carcinoma" "Invasive lobular carcinoma" "Invasive ductal carcinoma" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_behavior_cd        : int [1:1141] 3 3 3 3 3 3 3 3 3 3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_laterality_cd      : num [1:1141] 2 1 2 2 1 2 2 1 2 2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_grade              : int [1:1141] 2 1 9 2 2 3 3 3 1 3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_grade_diff             : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_grade                  : chr [1:1141] "II" "I" "" "II" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ best_ajcc_stage_cd        : chr [1:1141] "2A" "1A" "2B" "1A" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_stage_iv               : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_stage                  : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ stage_dx                  : chr [1:1141] "Stage II" "Stage I" "Stage II" "Stage I" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ stage_dx_iv               : chr [1:1141] "Stage I-III" "Stage I-III" "Stage I-III" "Stage I-III" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_tnm_path_desc      : int [1:1141] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_t_cd          : chr [1:1141] "P1C" "P1B" "P1C" "P1C" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t_stage           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t1_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t2_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t3_det            : logi [1:1141] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t4_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_n_cd          : chr [1:1141] "P1A" "P0" "P2A" "P0" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n_stage           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n1_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n2_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n3_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_m_cd          : chr [1:1141] "C0" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_stage_cd      : chr [1:1141] "2A" "1A" "3A" "1A" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_group_stage       : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_t_cd          : chr [1:1141] "CX" "C1A" "C2" "C1C" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t_stage           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t1_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t2_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t3_det            : logi [1:1141] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t4_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_n_cd          : chr [1:1141] "CX" "C0" "C1" "C0" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n_stage           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n1_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n2_det            : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clinical_n3_det        : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_m_cd          : chr [1:1141] "CX" "C0" "C0" "C0" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_stage_cd      : chr [1:1141] "99" "1A" "2B" "1A" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_seer_sum_stage     : int [1:1141] 3 1 3 1 3 7 3 NA 3 3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_tx_pre_path_stage      : chr [1:1141] "No" "No" "Yes" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_gen_sum_stage_2        : logi [1:1141] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_tnm_edition_num    : chr [1:1141] "6" "7" "7" "7" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_dmets_yn               : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets1           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets2           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets3           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets4           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets5           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets6           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets7           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets8           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets9           : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets10          : chr [1:1141] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_er                 : chr [1:1141] "Positive/elevated" "Positive/elevated" "Positive/elevated" "Positive/elevated" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_pr                 : chr [1:1141] "Positive/elevated" "Positive/elevated" "Positive/elevated" "Positive/elevated" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_her2ihc_val        : chr [1:1141] "Score of 2+" "Score 0" "Unknown or no information" "Score 0" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_her2ihc_intp       : chr [1:1141] "Positive/elevated" "Negative/normal" "Negative/normal" "Negative/normal" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_her_summ           : chr [1:1141] "Positive/elevated/amplified" "Negative/normal within normal limits" "Negative/normal within normal limits" "Negative/normal within normal limits" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_oncotypedx         : chr [1:1141] "Not documented in medical record" "10" "Not documented in medical record" "13" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_mgene              : chr [1:1141] "Not applicable" "Not applicable" "Not applicable" "Not applicable" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_mgeneresult        : chr [1:1141] "Not documented in medical record" "Not documented in medical record" "Not documented in medical record" "Not documented in medical record" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ bca_subtype               : chr [1:1141] "HER2+, HR+" "HER2-, HR+" "HER2-, HR+" "HER2-, HR+" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_n_regimens             : int [1:1141] 12 1 8 5 14 8 7 13 8 12 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int          : int [1:1141] 6284 2539 2053 1627 4129 813 3164 6734 2712 3264 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int_mos      : num [1:1141] 206.7 83.5 67.5 53.5 135.8 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int_yrs      : num [1:1141] 17.2 6.95 5.62 4.45 11.3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int              : int [1:1141] NA NA 2081 NA 4137 813 NA 6780 2714 3274 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int_mos          : num [1:1141] NA NA 68.5 NA 136.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int_yrs          : num [1:1141] NA NA 5.7 NA 11.3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_days             : int [1:1141] 6284 2539 2081 1627 4137 813 3164 6780 2714 3274 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_mos              : num [1:1141] 206.7 83.5 68.5 53.5 136.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_yrs              : num [1:1141] 17.2 6.95 5.7 4.45 11.33 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ os_status_dx              : int [1:1141] 0 0 1 0 1 1 0 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dmets_stage_i_iii         : int [1:1141] 1 0 1 0 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_days          : int [1:1141] 1503 NA 974 NA 1196 157 878 1303 2069 2003 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_mos           : num [1:1141] 49.4 NA 32 NA 39.3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_yrs           : num [1:1141] 4.11 NA 2.67 NA 3.27 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dmets_abdomen             : int [1:1141] 1 0 1 0 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_abdomen_days  : int [1:1141] 3981 NA 980 NA 4122 409 934 4426 2069 3168 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. .. [list output truncated]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ ca_dx_non_index   :'data.frame':	194 obs. of  103 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                  : chr [1:194] "BrCa" "BrCa" "BrCa" "BrCa" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id               : chr [1:194] "GENIE-DFCI-000275" "GENIE-DFCI-000570" "GENIE-DFCI-000732" "GENIE-DFCI-000911" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution             : chr [1:194] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_seq                  : int [1:194] 1 2 1 2 2 1 1 1 3 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ redcap_ca_index         : chr [1:194] "No" "" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_cadx_int             : int [1:194] 16767 NA 8920 19853 13710 10626 15528 16522 18403 14700 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_dx_how               : chr [1:194] "Pathology" "" "Pathology" "Pathology" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_next_ca_days        : int [1:194] 16786 NA 14956 NA NA 16723 17453 18403 NA 16617 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_first_index_ca       : int [1:194] -19 NA -6036 469 32 -6097 -1925 -1881 0 -1917 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_first_contact_int: int [1:194] 16781 NA 8920 NA 13737 10626 15584 16577 NA 17112 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_age                  : int [1:194] 45 NA 24 54 37 29 42 45 50 40 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ age_dx                  : int [1:194] 45 NA 24 54 37 29 42 45 50 40 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_site_cd          : chr [1:194] "C504" "" "C539" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_site                 : chr [1:194] "C50.9    Breast NOS" "" "C53.9    Cervix uteri" "C67.9    Bladder NOS" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_d_site               : chr [1:194] "C50.9" "" "C53.9" "C67.9" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ site_recode             : chr [1:194] "Breast" "" "Female Genital System" "Urinary System" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_type                 : chr [1:194] "Breast Cancer" "" "Cervical Cancer" "Bladder Cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_type_brain           : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_type_oth             : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_heme_malig           : chr [1:194] "" "" "" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_heme_type            : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_histology_cd     : int [1:194] 85003 NA 80772 NA 88323 80772 85003 84133 NA 85222 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_behavior_cd      : int [1:194] 3 NA 2 NA 3 2 3 3 NA 2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_laterality_cd    : num [1:194] 1 NA 0 NA 2 0 1 0 NA 2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_grade            : int [1:194] 9 NA 9 NA 9 9 3 3 NA 9 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_grade_diff           : chr [1:194] "" "" "" "III" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_grade                : chr [1:194] "" "" "" "III" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_stage_iv             : chr [1:194] "" "" "" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_stage                : chr [1:194] "" "" "" "Not Applicable" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ stage_dx_iv             : chr [1:194] "Stage I-III" "" "Stage 0" "Stage I-III" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_tnm_path_desc    : int [1:194] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_t_cd        : chr [1:194] "P1C" "" "PIS" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t_stage         : chr [1:194] "" "" "" "Unknown" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t1_det          : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t2_det          : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t3_det          : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t4_det          : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_n_cd        : chr [1:194] "P1A" "" "P0" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n_stage         : chr [1:194] "" "" "" "Unknown" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n1_det          : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n2_det          : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n3_det          : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_m_cd        : chr [1:194] "" "" "C0" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_stage_cd    : chr [1:194] "2A" "" "0" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_group_stage     : chr [1:194] "" "" "" "Unknown" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_t_cd        : chr [1:194] "C2" "" "CX" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t_stage         : chr [1:194] "" "" "" "Not Applicable" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t1_det          : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t2_det          : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t3_det          : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t4_det          : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_n_cd        : chr [1:194] "C1" "" "CX" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n_stage         : chr [1:194] "" "" "" "Not Applicable" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n1_det          : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n2_det          : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clinical_n3_det      : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_m_cd        : chr [1:194] "C0" "" "CX" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_stage_cd    : chr [1:194] "2B" "" "99" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_seer_sum_stage   : int [1:194] 3 NA NA NA NA NA 1 1 NA 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_tx_pre_path_stage    : chr [1:194] "Yes" "" "No" "Unknown" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_gen_sum_stage_2      : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_tnm_edition_num  : chr [1:194] "7" "" "4" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_dmets_yn             : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets1         : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets2         : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets3         : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets4         : chr [1:194] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets5         : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets6         : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets7         : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets8         : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets9         : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets10        : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_er               : chr [1:194] "Positive/elevated" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_pr               : chr [1:194] "Positive/elevated" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_her2ihc_val      : chr [1:194] "Unknown or no information" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_her2ihc_intp     : chr [1:194] "Negative/normal" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_her_summ         : chr [1:194] "Negative/normal within normal limits" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_oncotypedx       : chr [1:194] "Not documented in medical record" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_mgene            : chr [1:194] "Not applicable" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_bca_mgeneresult      : chr [1:194] "Not documented in medical record" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ bca_subtype             : chr [1:194] "HER2-, HR+" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_n_regimens           : int [1:194] 8 0 5 1 0 1 1 0 5 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int        : int [1:194] 2072 NA 8310 562 5935 10517 4570 5356 3475 2912 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int_mos    : num [1:194] 68.2 NA 273.4 18.5 195.2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int_yrs    : num [1:194] 5.67 NA 22.75 1.54 16.25 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int            : int [1:194] 2100 NA NA 564 6070 NA NA NA NA 3425 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int_mos        : num [1:194] 69.1 NA NA 18.6 199.7 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int_yrs        : num [1:194] 5.75 NA NA 1.54 16.62 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_days           : int [1:194] 2100 NA 8310 564 6070 10517 4570 5356 3475 3425 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_mos            : num [1:194] 69.1 NA 273.4 18.6 199.7 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_yrs            : num [1:194] 5.75 NA 22.75 1.54 16.62 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ os_status_dx            : int [1:194] 1 1 0 1 1 0 0 0 0 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_cohort              : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_i_adv_status        : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_adv_days       : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_adv_mos        : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_adv_yrs        : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_m_adv_status        : logi [1:194] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. .. [list output truncated]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ ca_drugs          :'data.frame':	6906 obs. of  102 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                          : chr [1:6906] "BrCa" "BrCa" "BrCa" "BrCa" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id                       : chr [1:6906] "GENIE-DFCI-097984" "GENIE-DFCI-086475" "GENIE-DFCI-086475" "GENIE-DFCI-086475" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution                     : chr [1:6906] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ regimen_number                  : int [1:6906] 1 1 2 3 4 1 1 2 2 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_seq                          : int [1:6906] 1 0 0 0 0 0 1 0 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ redcap_ca_index                 : chr [1:6906] "No" "Yes" "Yes" "Yes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_num                       : int [1:6906] 1 2 1 2 1 2 2 1 2 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_admin                     : chr [1:6906] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_inst                      : chr [1:6906] "At external institution only" "At the internal/native institution only" "At the internal/native institution only" "At the internal/native institution only" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_firstinst                 : chr [1:6906] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_ct_yn                     : chr [1:6906] "No" "No" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_dc_ynu                    : chr [1:6906] "Yes" "Yes" "Yes" "Yes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ regimen_drugs                   : chr [1:6906] "Doxorubicin Hydrochloride" "Doxorubicin Hydrochloride, Olaratumab" "Ifosfamide" "Docetaxel, Gemcitabine Hydrochloride" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_1                    : chr [1:6906] "Doxorubicin Hydrochloride(ADM,Adriacin,Adriamycin,Adriamycin Hydrochloride,Adriamycin RDF,Adriamycine,Adriblast"| __truncated__ "Doxorubicin Hydrochloride(ADM,Adriacin,Adriamycin,Adriamycin Hydrochloride,Adriamycin RDF,Adriamycine,Adriblast"| __truncated__ "Ifosfamide(Cyfos,Holoxan,Holoxane,IFO,IFOCell,IFX,Ifex,Ifolem,Ifomida,Ifomide,Ifosfamidum,Ifoxan,MJF9325,Mitoxa"| __truncated__ "Gemcitabine Hydrochloride(FF 10832,FF10832,FF10832,Gemzar,LY188011,LY188011,dFdCyd)" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_2                    : chr [1:6906] "" "Olaratumab(IMC3G3,Lartruvo)" "" "Docetaxel(Taxotere,RP56976)" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_3                    : chr [1:6906] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_4                    : chr [1:6906] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_5                    : chr [1:6906] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_1                : chr [1:6906] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_2                : chr [1:6906] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_3                : logi [1:6906] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_4                : logi [1:6906] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_5                : logi [1:6906] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_1             : int [1:6906] 5774 7427 7470 7491 7573 8151 8167 8235 8272 8305 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_2             : int [1:6906] NA 7427 NA 7491 NA 8151 8167 NA 8272 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_3             : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_4             : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_5             : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_1               : int [1:6906] 6139 7455 7473 7526 7707 8214 8209 9162 8623 8670 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_2               : int [1:6906] NA 7455 NA 7491 NA 8214 8216 NA 8623 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_3               : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_4               : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_5               : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_1              : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_2              : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_3              : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_4              : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_5              : logi [1:6906] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_1             : int [1:6906] 30 732 775 796 878 101 1155 185 1260 15 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_2             : int [1:6906] NA 732 NA 796 NA 101 1155 NA 1260 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_3             : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_4             : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_5             : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_1         : num [1:6906] 0.987 24.079 25.493 26.184 28.882 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_2         : num [1:6906] NA 24.1 NA 26.2 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_3         : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_4         : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_5         : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_1            : int [1:6906] 366 29 4 36 135 64 43 928 352 366 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_2            : int [1:6906] NA 29 NA 1 NA 64 50 NA 352 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_3            : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_4            : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_5            : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_1 : int [1:6906] 366 29 4 36 135 64 43 928 352 366 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_2 : int [1:6906] NA 29 NA 1 NA 64 50 NA 352 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_3 : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_4 : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_5 : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_reg_start_int                : int [1:6906] 30 732 775 796 878 101 1155 185 1260 15 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_reg_start_int_mos            : num [1:6906] 0.987 24.079 25.493 26.184 28.882 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_reg_start_int_yrs            : num [1:6906] 0.0821 2.0041 2.1218 2.1793 2.4038 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_any_int           : int [1:6906] 366 29 4 1 135 64 43 928 352 366 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_any_int_mos       : num [1:6906] 12.0395 0.9539 0.1316 0.0329 4.4408 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_any_int_yrs       : num [1:6906] 1.00205 0.0794 0.01095 0.00274 0.36961 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_any_int: int [1:6906] 366 29 4 1 135 64 43 928 352 366 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_any_mos: num [1:6906] 12.0395 0.9539 0.1316 0.0329 4.4408 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_any_yrs: num [1:6906] 1.00205 0.0794 0.01095 0.00274 0.36961 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_all_int           : int [1:6906] 366 29 4 36 135 64 50 928 352 366 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_all_int_mos       : num [1:6906] 12.039 0.954 0.132 1.184 4.441 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_all_int_yrs       : num [1:6906] 1.0021 0.0794 0.011 0.0986 0.3696 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_all_int: int [1:6906] 366 29 4 36 135 64 50 928 352 366 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_all_mos: num [1:6906] 12.039 0.954 0.132 1.184 4.441 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_all_yrs: num [1:6906] 1.0021 0.0794 0.011 0.0986 0.3696 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d1_days                   : int [1:6906] 8317 409 366 345 263 7747 4838 7663 4733 10368 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d2_days                   : int [1:6906] NA 409 NA 345 NA 7747 4838 NA 4733 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d3_days                   : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d4_days                   : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d5_days                   : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d1_mos                    : num [1:6906] 273.59 13.45 12.04 11.35 8.65 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d2_mos                    : num [1:6906] NA 13.5 NA 11.3 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d3_mos                    : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d4_mos                    : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d5_mos                    : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d1_yrs                    : num [1:6906] 22.771 1.12 1.002 0.945 0.72 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d2_yrs                    : num [1:6906] NA 1.12 NA 0.945 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d3_yrs                    : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d4_yrs                    : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d5_yrs                    : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ os_status_g                     : int [1:6906] 1 1 1 1 1 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ os_status_d                     : int [1:6906] 1 1 1 1 1 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_g_days                    : int [1:6906] 8317 409 366 345 263 7747 4838 7663 4733 10368 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_g_mos                     : num [1:6906] 273.59 13.45 12.04 11.35 8.65 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_g_yrs                     : num [1:6906] 22.771 1.12 1.002 0.945 0.72 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_i_status_g                  : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_g_days                 : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_g_mos                  : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_g_yrs                  : num [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_m_status_g                  : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_m_g_days                 : int [1:6906] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. .. [list output truncated]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ prissmm_imaging   :'data.frame':	26763 obs. of  43 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort            : chr [1:26763] "BrCa" "BrCa" "BrCa" "BrCa" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id         : chr [1:26763] "GENIE-DFCI-000183" "GENIE-DFCI-000183" "GENIE-DFCI-000183" "GENIE-DFCI-000183" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution       : chr [1:26763] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ scan_number       : int [1:26763] 1 2 3 4 5 6 7 8 9 10 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scan_int    : int [1:26763] 13206 13207 13579 13962 14333 14691 14692 14697 14697 14697 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_scan_days      : int [1:26763] 5 6 378 761 1132 1490 1491 1496 1496 1496 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_scan_mos       : num [1:26763] 0.164 0.197 12.434 25.033 37.237 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_scan_yrs       : num [1:26763] 0.0137 0.0164 1.0349 2.0835 3.0992 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_ref_scan_int: int [1:26763] 13201 NA NA NA NA 14333 NA 13238 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_ref_scan_days  : int [1:26763] 0 NA NA NA NA 1132 NA 37 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_ref_scan_mos   : num [1:26763] 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_ref_scan_yrs   : num [1:26763] 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_inst_perf   : chr [1:26763] "External institution" "External institution" "Internal institution" "Internal institution" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_inst_inter  : chr [1:26763] "External institution" "External institution" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scan_type   : chr [1:26763] "MRI" "MRI" "Mammogram - Use for Breast Cancer only" "Mammogram - Use for Breast Cancer only" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ oth_scan          : logi [1:26763] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___1: chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___2: chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___3: chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___4: chr [1:26763] "Chest" "Chest" "Chest" "Chest" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___5: chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___6: chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___7: chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___8: chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ scan_sites        : chr [1:26763] "Chest" "Chest" "Chest" "Chest" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_ca          : chr [1:26763] "Yes, the Impression states or implies there is evidence of cancer" "No, the Impression states or implies there is no evidence of cancer" "No, the Impression states or implies there is no evidence of cancer" "No, the Impression states or implies there is no evidence of cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_overall     : chr [1:26763] "Not stated/Indeterminate" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite1     : chr [1:26763] "C50.9    Breast NOS" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite2     : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite3     : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite4     : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite5     : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite6     : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite7     : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite8     : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite9     : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite10    : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite11    : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite12    : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite13    : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite14    : chr [1:26763] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite15    : logi [1:26763] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version   : num [1:26763] 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ prissmm_pathology :'data.frame':	7223 obs. of  371 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort               : chr [1:7223] "BrCa" "BrCa" "BrCa" "BrCa" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id            : chr [1:7223] "GENIE-DFCI-000183" "GENIE-DFCI-000183" "GENIE-DFCI-000183" "GENIE-DFCI-000183" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution          : chr [1:7223] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_number     : int [1:7223] 1 2 3 4 5 6 7 1 2 3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_rep_number      : int [1:7223] 1 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_inst       : chr [1:7223] "External institution" "Internal institution" "Internal institution" "Internal institution" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_rep_inst        : chr [1:7223] "External institution" "Internal institution" "Internal institution" "Internal institution" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_int        : int [1:7223] 13201 13209 14174 14692 14712 16024 19175 15959 16035 16750 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_days    : int [1:7223] 0 8 973 1491 1511 2823 5974 0 76 791 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_mos     : num [1:7223] 0 0.263 32.007 49.046 49.704 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_yrs     : num [1:7223] 0 0.0219 2.6639 4.0821 4.1369 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_type       : chr [1:7223] "Surgical pathology" "Surgical pathology" "Surgical pathology" "Surgical pathology" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_num_spec        : int [1:7223] 1 9 1 1 1 1 1 2 8 2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site1           : chr [1:7223] "C50.8    Overlapping lesion of breast" "C50.9    Breast NOS" "C54.1    Endometrium" "C50.9    Breast NOS" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site2           : chr [1:7223] "" "C50.9    Breast NOS" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site3           : chr [1:7223] "" "C50.9    Breast NOS" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site4           : chr [1:7223] "" "C50.9    Breast NOS" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site5           : chr [1:7223] "" "C50.9    Breast NOS" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site6           : chr [1:7223] "" "C50.9    Breast NOS" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site7           : chr [1:7223] "" "C50.9    Breast NOS" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site8           : chr [1:7223] "" "C77.3    Lymph nodes of axilla or arm" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site9           : chr [1:7223] "" "C77.3    Lymph nodes of axilla or arm" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site10          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site11          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site12          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site13          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site14          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site15          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site16          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site17          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site18          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site19          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site20          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site21          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site22          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site23          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site24          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site25          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site26          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site27          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site28          : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site29          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site30          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site31          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site32          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site33          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site34          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site35          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site36          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site37          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site38          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site39          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site40          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site41          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site42          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site43          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site44          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site45          : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu_any      : chr [1:7223] "Yes" "Yes" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_specimen_insitu    : int [1:7223] 1 1 0 0 0 0 0 1 1 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu1         : chr [1:7223] "Yes" "Yes" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu2         : chr [1:7223] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu3         : chr [1:7223] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu4         : chr [1:7223] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu5         : chr [1:7223] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu6         : chr [1:7223] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu7         : chr [1:7223] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu8         : chr [1:7223] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu9         : chr [1:7223] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu10        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu11        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu12        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu13        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu14        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu15        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu16        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu17        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu18        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu19        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu20        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu21        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu22        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu23        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu24        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu25        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu26        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu27        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu28        : chr [1:7223] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu29        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu30        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu31        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu32        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu33        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu34        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu35        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu36        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu37        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu38        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu39        : logi [1:7223] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. .. [list output truncated]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ prissmm_md        :'data.frame':	28293 obs. of  12 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort          : chr [1:28293] "BrCa" "BrCa" "BrCa" "BrCa" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id       : chr [1:28293] "GENIE-DFCI-000183" "GENIE-DFCI-000183" "GENIE-DFCI-000183" "GENIE-DFCI-000183" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution     : chr [1:28293] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_visit_number : int [1:28293] 1 2 3 4 5 6 7 8 9 10 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_onc_visit_int: int [1:28293] 13216 13257 13299 13313 13341 13453 13579 13787 13937 13962 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_md_visit_days: int [1:28293] 15 56 98 112 140 252 378 586 736 761 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_md_visit_mos : num [1:28293] 0.493 1.842 3.224 3.684 4.605 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_md_visit_yrs : num [1:28293] 0.0411 0.1533 0.2683 0.3066 0.3833 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_type_ca_cur  : chr [1:28293] "Breast Cancer" "Breast Cancer" "Breast Cancer" "Breast Cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_ca           : chr [1:28293] "No, the Impression/Plan states or implies there is no evidence of cancer" "No, the Impression/Plan states or implies there is no evidence of cancer" "No, the Impression/Plan states or implies there is no evidence of cancer" "No, the Impression/Plan states or implies there is no evidence of cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_ca_status    : chr [1:28293] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version : num [1:28293] 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ tumor_marker      :'data.frame':	9744 obs. of  14 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort               : chr [1:9744] "BrCa" "BrCa" "BrCa" "BrCa" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id            : chr [1:9744] "GENIE-DFCI-000183" "GENIE-DFCI-000183" "GENIE-DFCI-000183" "GENIE-DFCI-000183" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution          : chr [1:9744] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_number            : int [1:9744] 1 2 3 4 5 6 7 8 9 10 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_spec_collect_int  : int [1:9744] 15383 15441 15497 15524 15553 15581 15614 15644 15677 15706 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_tm_days           : int [1:9744] 2182 2240 2296 2323 2352 2380 2413 2443 2476 2505 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_tm_mos            : num [1:9744] 71.8 73.7 75.5 76.4 77.4 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_tm_yrs            : num [1:9744] 5.97 6.13 6.29 6.36 6.44 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_type              : chr [1:9744] "CA2729" "CA2729" "CA2729" "CA2729" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_num_result        : num [1:9744] 12 10 9 10 10 10 9 8 9 10 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_result_units      : chr [1:9744] "U/mL" "U/mL" "U/mL" "U/mL" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_normal_range_lower: int [1:9744] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_normal_range_upper: num [1:9744] 38 38 38 38 38 38 38 38 38 38 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version      : num [1:9744] 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ cpt               :'data.frame':	1234 obs. of  27 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                : chr [1:1234] "BrCa" "BrCa" "BrCa" "BrCa" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id             : chr [1:1234] "GENIE-DFCI-000183" "GENIE-DFCI-000226" "GENIE-DFCI-000275" "GENIE-DFCI-000284" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution           : chr [1:1234] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_number            : int [1:1234] 1 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_report_int        : int [1:1234] 16842 16087 17979 17123 13979 15546 16991 16461 20209 19938 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_seq                : int [1:1234] 0 0 2 0 0 0 0 0 0 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_number      : int [1:1234] 6 2 4 1 4 4 4 4 2 4 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_rep_number       : int [1:1234] 1 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_genie_sample_id   : chr [1:1234] "GENIE-DFCI-000183-1356" "GENIE-DFCI-000226-1542" "GENIE-DFCI-000275-9265" "GENIE-DFCI-000284-9288" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_oncotree_code     : chr [1:1234] "BRCA" "MDLC" "ILC" "IDC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_sample_type       : chr [1:1234] "4" "1" "1" "1" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ sample_type           : chr [1:1234] "Metastasis site unspecified" "Primary tumor" "Primary tumor" "Primary tumor" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_seq_assay_id      : chr [1:1234] "DFCI-ONCOPANEL-1" "DFCI-ONCOPANEL-1" "DFCI-ONCOPANEL-2" "DFCI-ONCOPANEL-2" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_order_int         : int [1:1234] 16770 16048 17931 17066 13951 15520 16940 16394 20171 19913 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_seq_date          : int [1:1234] 2014 2014 2016 2016 2016 2016 2016 2013 2014 2016 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_cpt_report_days   : int [1:1234] 16842 16087 17979 17123 13979 15546 16991 16461 20209 19938 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_report_post_death : int [1:1234] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_cpt_rep_days       : int [1:1234] 3641 128 1193 128 2637 260 1520 6151 2268 2378 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_cpt_rep_mos        : num [1:1234] 119.77 4.21 39.24 4.21 86.74 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_cpt_rep_yrs        : num [1:1234] 9.97 0.35 3.27 0.35 7.22 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_cpt_days : int [1:1234] 2823 76 141 0 2499 176 878 1303 26 2248 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_cpt_mos  : num [1:1234] 92.86 2.5 4.64 0 82.2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_cpt_yrs  : num [1:1234] 7.729 0.208 0.386 0 6.842 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_cpt_rep_days: int [1:1234] 818 52 1052 128 138 84 642 4848 2242 130 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_cpt_rep_mos : num [1:1234] 26.91 1.71 34.61 4.21 4.54 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_cpt_rep_yrs : num [1:1234] 2.24 0.142 2.88 0.35 0.378 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version       : num [1:1234] 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 1.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ fusions           :'data.frame':	610 obs. of  9 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Hugo_Symbol         : chr [1:610] "TMPRSS2" "TMPRSS2" "NOTCH3" "AKT2" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Entrez_Gene_Id      : int [1:610] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Center              : chr [1:610] "MSK" "MSK" "MSK" "MSK" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Sample_Barcode: chr [1:610] "GENIE-MSK-P-0000224-T01-IM3" "GENIE-MSK-P-0000224-T01-IM3" "GENIE-MSK-P-0001308-T01-IM3" "GENIE-MSK-P-0001760-T01-IM3" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Fusion              : chr [1:610] "WFDC3-TMPRSS2 fusion" "RIPK4-TMPRSS2 fusion" "NOTCH3-intragenic" "AKT2-ACTN4 fusion" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ DNA_support         : chr [1:610] "yes" "yes" "yes" "yes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ RNA_support         : chr [1:610] "unknown" "unknown" "unknown" "unknown" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Method              : chr [1:610] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Frame               : chr [1:610] "unknown" "in frame" "unknown" "in frame" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ cna               :'data.frame':	930 obs. of  1222 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Hugo_Symbol                : chr [1:930] "ABCB11" "ABL1" "ABL2" "ACTB" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001738.6623     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001665.1119     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.009803.7618     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008762.7581     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.003186.86       : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002294.10756    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011618.10772    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011577.10596    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001859.10618    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.000570.10701    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.009660.10902    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011699.10666    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.000290.11285    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010991.11464    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008810.11270    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002780.10910    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.003170.11295    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002871.11453    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011105.11120    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.007525.11157    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011113.11155    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001510.10211    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011000.11226    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011438.11187    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.009377.11590    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001931.10941    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010998.11477    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011083.11367    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001527.11768    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.004278.979      : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002668.980      : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001619.1008     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011531.11692    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011198.11695    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008412.10407    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002487.11750    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002448.11734    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011219.11544    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001986.11553    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.013720.22897    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.015315.23691    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.023685.23971    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.020359.23927    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.012102.23934    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.023720.23862    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.023636.23861    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037096.35639    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002929.2112     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.035015.36093    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001476.2139     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.034842.48558    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037362.48519    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.024185.61060    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.027183.61539    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037831.61540    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037848.61554    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.038007.61387    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.038024.61388    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037836.61427    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.031751.74169    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037318.74339    : int [1:930] 0 0 NA NA NA -1 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037736.74408    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.034334.74259    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037718.74228    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037310.74246    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.036145.74614    : int [1:930] 1 1 NA NA NA 1 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.025659.74621    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002725.1814     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.024103.74611    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.036441.74673    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.027051.74640    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037367.74686    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.034058.74877    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.033935.74929    : int [1:930] -1 -1 NA NA NA -1 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.026152.74971    : int [1:930] 0 1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037091.75002    : int [1:930] 0 1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037885.74958    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.027436.75006    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.033107.74829    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.036755.74884    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.034513.74907    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037851.74910    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037851.250051   : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.036431.203492   : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.027496.88604    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.027496.159240   : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.027167.88607    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.026427.88556    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.025894.88547    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.036811.88538    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.027852.88467    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.036617.88502    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.036744.88490    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037000.116490   : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.036372.116389   : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037447.116448   : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.032900.130496   : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037973.130282   : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. .. [list output truncated]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ mutations_extended:'data.frame':	6633 obs. of  54 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Hugo_Symbol                  : chr [1:6633] "NBN" "GATA6" "MAP3K1" "TERT" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Entrez_Gene_Id               : int [1:6633] 4683 2627 4214 7015 8289 83596 25766 2033 2064 2068 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Center                       : chr [1:6633] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ NCBI_Build                   : chr [1:6633] "GRCh37" "GRCh37" "GRCh37" "GRCh37" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Chromosome                   : chr [1:6633] "8" "18" "5" "5" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Start_Position               : int [1:6633] 90983459 19751478 56178169 1278911 27023623 50173702 50037528 41542773 37880981 45867712 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ End_Position                 : int [1:6633] 90983459 19751478 56178178 1278911 27023632 50173702 50037528 41542773 37880982 45867712 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Strand                       : chr [1:6633] "+" "+" "+" "+" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Variant_Classification       : chr [1:6633] "Missense_Mutation" "Missense_Mutation" "Frame_Shift_Del" "Missense_Mutation" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Variant_Type                 : chr [1:6633] "SNP" "SNP" "DEL" "SNP" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Reference_Allele             : chr [1:6633] "C" "A" "CAGTCAAGAC" "C" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Seq_Allele1            : chr [1:6633] "C" "A" "CAGTCAAGAC" "C" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Seq_Allele2            : chr [1:6633] "T" "G" "-" "A" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dbSNP_RS                     : chr [1:6633] "rs61753718" "rs759976914" "novel" "novel" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dbSNP_Val_Status             : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Sample_Barcode         : chr [1:6633] "GENIE-DFCI-000183-1356" "GENIE-DFCI-000183-1356" "GENIE-DFCI-000183-1356" "GENIE-DFCI-000183-1356" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Matched_Norm_Sample_Barcode  : chr [1:6633] "NORMAL" "NORMAL" "NORMAL" "NORMAL" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Match_Norm_Seq_Allele1       : int [1:6633] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Match_Norm_Seq_Allele2       : int [1:6633] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Validation_Allele1     : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Validation_Allele2     : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Match_Norm_Validation_Allele1: logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Match_Norm_Validation_Allele2: logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Verification_Status          : chr [1:6633] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Validation_Status            : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Mutation_Status              : chr [1:6633] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Sequencing_Phase             : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Sequence_Source              : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Validation_Method            : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Score                        : chr [1:6633] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ BAM_File                     : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Sequencer                    : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ HGVSp_Short                  : chr [1:6633] "p.R215Q" "p.S125G" "p.Q1048Pfs*31" "p.V711L" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ t_ref_count                  : int [1:6633] 94 17 289 64 53 16 23 19 262 99 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ t_alt_count                  : int [1:6633] 149 56 129 39 28 8 31 42 21 77 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_ref_count                  : int [1:6633] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_alt_count                  : int [1:6633] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Protein_position             : chr [1:6633] "215/754" "125/595" "1048-1051/1512" "711/1132" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Codons                       : chr [1:6633] "cGg/cAg" "Agc/Ggc" "CAGTCAAGACcc/cc" "Gtg/Ttg" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ SWISSPROT                    : chr [1:6633] "O60934" "Q92908" "Q13233" "O14746" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ RefSeq                       : chr [1:6633] "NM_002485.4" "NM_005257.4" "NM_005921.1" "NM_198253.2,NM_001193376.1" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ t_depth                      : int [1:6633] 243 73 418 103 81 24 54 61 283 176 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_depth                      : int [1:6633] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ FILTER                       : chr [1:6633] "PASS" "PASS" "PASS" "PASS" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_AF                    : num [1:6633] 2.44e-05 4.40e-06 NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_AFR_AF                : num [1:6633] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_AMR_AF                : num [1:6633] NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_ASJ_AF                : num [1:6633] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_EAS_AF                : num [1:6633] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_FIN_AF                : num [1:6633] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_NFE_AF                : num [1:6633] 4.49e-05 9.73e-06 NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_OTH_AF                : num [1:6633] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_SAS_AF                : num [1:6633] 3.25e-05 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ mutationInCis_Flag           : logi [1:6633] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  $ CRC_v1.2 :List of 12</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ pt_char           :'data.frame':	1498 obs. of  40 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                    : chr [1:1498] "CRC" "CRC" "CRC" "CRC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id                 : chr [1:1498] "GENIE-DFCI-111551" "GENIE-DFCI-111483" "GENIE-DFCI-111479" "GENIE-DFCI-110658" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution               : chr [1:1498] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ redacted                  : chr [1:1498] "No" "No" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ birth_year                : int [1:1498] 1968 1962 1946 1967 1985 1953 1973 1988 1957 1966 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_ethnicity_code     : chr [1:1498] "Non-Spanish; non-Hispanic" "Non-Spanish; non-Hispanic" "Non-Spanish; non-Hispanic" "Non-Spanish; non-Hispanic" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_race_code_primary  : chr [1:1498] "White" "Black" "White" "White" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_race_code_secondary: chr [1:1498] "No further race documented" "No further race documented" "No further race documented" "No further race documented" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_race_code_tertiary : chr [1:1498] "No further race documented" "No further race documented" "No further race documented" "No further race documented" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_sex_code           : chr [1:1498] "Female" "Male" "Female" "Male" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ hybrid_death_int          : int [1:1498] NA NA NA NA NA NA NA NA 22191 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ hybrid_death_source       : chr [1:1498] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ last_oncvisit_int         : int [1:1498] 18790 20777 26934 19179 11666 24462 16763 11543 22169 18782 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ last_alive_int            : int [1:1498] 18790 20931 26934 19275 11666 24543 16993 11744 22169 18784 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ last_anyvisit_int         : int [1:1498] 18790 20931 26934 19275 11666 24462 16763 11543 22169 18837 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ enroll_hospice_int        : int [1:1498] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_lastalive_int         : int [1:1498] 18790 20931 26934 19275 11666 24543 16993 11744 22169 18837 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_lastalive_int_mos     : num [1:1498] 618 689 886 634 384 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_lastalive_int_yrs     : num [1:1498] 51.4 57.3 73.7 52.8 31.9 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ age_death_yrs             : int [1:1498] NA NA NA NA NA NA NA NA 60 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ age_last_fu_yrs           : int [1:1498] 51 57 73 52 31 67 46 32 60 51 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_cancers                 : int [1:1498] 1 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_cancers_index           : int [1:1498] 1 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_regimens_pt             : int [1:1498] 2 1 2 0 1 3 2 4 1 2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_imaging_reports_pt      : int [1:1498] 18 17 15 8 4 23 11 18 9 8 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_ct_pt             : int [1:1498] 17 16 14 3 3 19 10 13 4 7 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_mri_pt            : int [1:1498] 0 0 0 5 0 3 1 4 2 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_pet_ct_pt         : int [1:1498] 1 1 1 0 1 1 0 1 3 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_mammog_pt         : int [1:1498] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_bone_pt           : int [1:1498] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_scans_other_pt          : int [1:1498] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_path_reports_pt         : int [1:1498] 3 7 2 2 2 5 3 5 2 2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_md_notes_pt             : int [1:1498] 26 12 28 1 3 19 12 17 3 6 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_pt                   : int [1:1498] 39 5 35 0 0 22 7 1 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_ca15_3_pt            : int [1:1498] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_ca19_9_pt            : int [1:1498] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_ca2729_pt            : int [1:1498] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_tm_cea_pt               : int [1:1498] 39 5 35 0 0 22 7 1 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_cpt_pt                  : int [1:1498] 1 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version           : num [1:1498] 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ ca_dx_index       :'data.frame':	1508 obs. of  152 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                    : chr [1:1508] "CRC" "CRC" "CRC" "CRC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id                 : chr [1:1508] "GENIE-DFCI-000048" "GENIE-DFCI-000147" "GENIE-DFCI-000233" "GENIE-DFCI-000247" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution               : chr [1:1508] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_seq                    : int [1:1508] 0 0 2 1 0 1 0 0 1 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ redcap_ca_index           : chr [1:1508] "Yes" "Yes" "Yes" "Yes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_cadx_int               : int [1:1508] 21714 18521 17190 23359 20372 29978 17727 16175 30290 16954 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_dx_how                 : chr [1:1508] "Pathology" "Pathology" "Pathology" "Pathology" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_next_ca_days          : int [1:1508] NA NA NA 23413 NA 31672 NA NA 30996 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_first_index_ca         : int [1:1508] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_first_contact_int  : int [1:1508] 21765 18654 19076 23553 20443 29994 17782 16227 30536 16981 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_age                    : int [1:1508] 59 50 47 63 55 82 48 44 82 46 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ age_dx                    : int [1:1508] 59 50 47 63 55 82 48 44 82 46 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_site_cd            : chr [1:1508] "C180" "C209" "C209" "C209" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_site                   : chr [1:1508] "C18.0    Cecum" "C20.9    Rectum NOS" "C20.9    Rectum NOS" "C20.9    Rectum NOS" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_d_site                 : chr [1:1508] "C18.0" "C20.9" "C20.9" "C20.9" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ site_recode               : chr [1:1508] "Colon and Rectum" "Colon and Rectum" "Colon and Rectum" "Colon and Rectum" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_type                   : chr [1:1508] "Colon Cancer" "Rectum and Rectosigmoid Cancer" "Rectum and Rectosigmoid Cancer" "Rectum and Rectosigmoid Cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_histology_cd       : int [1:1508] 81403 81403 81403 81403 81403 81403 84803 81403 81403 82103 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_hist_adeno_squamous    : chr [1:1508] "Adenocarcinoma" "Adenocarcinoma" "Adenocarcinoma" "Adenocarcinoma" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_behavior_cd        : int [1:1508] 3 3 3 3 3 3 3 3 3 3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_laterality_cd      : num [1:1508] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_grade              : int [1:1508] 2 2 2 1 2 2 2 2 2 9 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_grade_diff             : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_grade                  : chr [1:1508] "II" "II" "II" "I" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ best_ajcc_stage_cd        : chr [1:1508] "2A" "2A" "" "4A" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_stage_iv               : chr [1:1508] "" "" "No" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_stage                  : chr [1:1508] "" "" "Unknown" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ stage_dx                  : chr [1:1508] "Stage II" "Stage II" "Stage III" "Stage IV" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ stage_dx_iv               : chr [1:1508] "Stage I-III" "Stage I-III" "Stage I-III" "Stage IV" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_tnm_path_desc      : num [1:1508] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_t_cd          : chr [1:1508] "P3" "P2" "PX" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t_stage           : chr [1:1508] "" "" "T1" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t1_det            : chr [1:1508] "" "" "Unknown" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t2_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t3_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t4_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_n_cd          : chr [1:1508] "P0" "P0" "PX" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n_stage           : chr [1:1508] "" "" "N1" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n1_det            : chr [1:1508] "" "" "Unknown" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n2_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n3_det            : logi [1:1508] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_m_cd          : chr [1:1508] "" "" "PX" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_stage_cd      : chr [1:1508] "2A" "1" "99" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_group_stage       : chr [1:1508] "" "" "III" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_t_cd          : chr [1:1508] "C3" "C3" "CX" "C3" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t_stage           : chr [1:1508] "" "" "Unknown" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t1_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t2_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t3_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t4_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_n_cd          : chr [1:1508] "C0" "C0" "CX" "CX" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n_stage           : chr [1:1508] "" "" "Unknown" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n1_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n2_det            : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clinical_n3_det        : logi [1:1508] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_m_cd          : chr [1:1508] "C0" "C0" "CX" "C1A" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_stage_cd      : chr [1:1508] "2A" "2A" "99" "4A" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_seer_sum_stage     : int [1:1508] 1 1 9 7 7 2 4 4 9 7 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_tx_pre_path_stage      : chr [1:1508] "" "" "Yes" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_gen_sum_stage_2        : logi [1:1508] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_tnm_edition_num    : chr [1:1508] "7" "7" "6" "7" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_dmets_yn               : chr [1:1508] "" "" "" "Yes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets1           : chr [1:1508] "" "" "" "C22.0    Liver" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets2           : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets3           : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets4           : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets5           : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets6           : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets7           : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets8           : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets9           : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets10          : chr [1:1508] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_crc_td                 : chr [1:1508] "0" "0" "Not documented in medical record" "Not applicable: Information not collected for this case" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_crc_crm                : chr [1:1508] "20" "5" "16" "Not applicable" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_crc_peri_inv           : chr [1:1508] "Perineural invasion present" "None" "None" "Not applicable" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_n_regimens             : int [1:1508] 1 4 6 3 5 0 3 1 6 8 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int          : int [1:1508] 324 1172 3292 1146 1230 1773 1623 2039 723 809 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int_mos      : num [1:1508] 10.7 38.6 108.3 37.7 40.5 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int_yrs      : num [1:1508] 0.887 3.209 9.013 3.138 3.368 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int              : int [1:1508] NA 1235 3303 1163 1230 NA NA NA 817 814 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int_mos          : num [1:1508] NA 40.6 108.7 38.3 40.5 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int_yrs          : num [1:1508] NA 3.38 9.04 3.18 3.37 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_days             : int [1:1508] 324 1235 3303 1163 1230 1773 1623 2039 817 814 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_mos              : num [1:1508] 10.7 40.6 108.7 38.3 40.5 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_yrs              : num [1:1508] 0.887 3.381 9.043 3.184 3.368 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ os_status_dx              : int [1:1508] 0 1 1 1 1 0 0 0 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dmets_stage_i_iii         : int [1:1508] 0 1 1 NA NA 1 1 0 1 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_days          : int [1:1508] NA 445 140 NA NA 1679 146 NA 227 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_mos           : num [1:1508] NA 14.64 4.61 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_yrs           : num [1:1508] NA 1.218 0.383 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dmets_abdomen             : int [1:1508] 0 0 1 0 1 0 0 0 0 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_abdomen_days  : int [1:1508] NA NA 2381 NA 1180 NA NA NA NA 6 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_abdomen_mos   : num [1:1508] NA NA 78.3 NA 38.8 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_abdomen_yrs   : num [1:1508] NA NA 6.52 NA 3.23 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dmets_bone                : int [1:1508] 0 1 1 0 1 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_bone_days     : int [1:1508] NA 1148 2912 NA 1091 NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_bone_mos      : num [1:1508] NA 37.8 95.8 NA 35.9 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_to_dmets_bone_yrs      : num [1:1508] NA 3.14 7.97 NA 2.99 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dmets_brain               : int [1:1508] 0 0 0 0 1 0 0 0 1 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. .. [list output truncated]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ ca_dx_non_index   :'data.frame':	346 obs. of  97 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                  : chr [1:346] "CRC" "CRC" "CRC" "CRC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id               : chr [1:346] "GENIE-DFCI-000233" "GENIE-DFCI-000247" "GENIE-DFCI-000311" "GENIE-DFCI-000946" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution             : chr [1:346] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_seq                  : int [1:346] 1 2 2 2 1 1 2 1 3 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ redcap_ca_index         : chr [1:346] "No" "No" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_cadx_int             : int [1:346] 14926 23413 31672 30996 23943 12513 31832 18001 27081 20188 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_dx_how               : chr [1:346] "Pathology" "Pathology" "Pathology" "Pathology" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_next_ca_days        : int [1:346] 17190 NA NA NA 27467 18522 NA 27081 NA 23282 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_first_index_ca       : int [1:346] -2264 54 1694 706 -3524 -6009 1301 -9080 0 -3094 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_first_contact_int: int [1:346] 14926 NA 31721 30991 23971 12508 NA 18016 NA 20188 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_age                  : int [1:346] 40 64 86 84 65 34 87 49 74 55 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ age_dx                  : int [1:346] 40 64 86 84 65 34 87 49 74 55 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_site_cd          : chr [1:346] "C539" "" "C508" "C712" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_site                 : chr [1:346] "C53.9    Cervix uteri" "C37.9    Thymus" "C50.8    Overlapping lesion of breast" "C71.2    Temporal lobe" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_d_site               : chr [1:346] "C53.9" "C37.9" "C50.8" "C71.2" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ site_recode             : chr [1:346] "Female Genital System" "Endocrine System" "Breast" "Brain and Other Nervous System" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_type                 : chr [1:346] "Uterus Cancer" "Thymus Cancer" "Breast Cancer" "Brain Cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_type_brain           : chr [1:346] "" "" "" "Astrocytic Tumors" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_type_oth             : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_heme_malig           : chr [1:346] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_heme_type            : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_histology_cd     : int [1:346] 80702 NA 85003 94403 81403 94013 NA 80723 NA 81403 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_behavior_cd      : int [1:346] 2 NA 3 3 3 3 NA 3 NA 3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_laterality_cd    : num [1:346] 0 NA 1 2 0 0 NA 0 NA 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_grade            : int [1:346] 9 NA NA 9 2 4 NA 3 NA 2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_grade_diff           : chr [1:346] "" "Grade/differentiation unknown or not stated or not applicable" "II" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_grade                : chr [1:346] "" "" "II" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_stage_iv             : chr [1:346] "" "No" "No" "Unknown" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_stage                : chr [1:346] "" "Not Applicable" "IA" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ stage_dx_iv             : chr [1:346] "Stage 0" "Stage I-III" "Stage I-III" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_tnm_path_desc    : num [1:346] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_t_cd        : chr [1:346] "PIS" "" "PT1C" "88" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t_stage         : chr [1:346] "" "Unknown" "T1" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t1_det          : chr [1:346] "" "" "T1c" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t2_det          : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t3_det          : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_t4_det          : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_n_cd        : chr [1:346] "P0" "" "PNX" "88" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n_stage         : chr [1:346] "" "Unknown" "N0" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n1_det          : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n2_det          : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_n3_det          : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_m_cd        : chr [1:346] "C0" "" "CM0" "88" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_path_stage_cd    : chr [1:346] "0" "" "99" "88" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_path_group_stage     : chr [1:346] "" "IIA" "IA" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_t_cd        : chr [1:346] "CX" "" "CT1C" "88" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t_stage         : chr [1:346] "" "Not Applicable" "Not Applicable" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t1_det          : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t2_det          : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t3_det          : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_t4_det          : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_n_cd        : chr [1:346] "CX" "" "CN0" "88" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n_stage         : chr [1:346] "" "Not Applicable" "Not Applicable" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n1_det          : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clin_n2_det          : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_clinical_n3_det      : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_m_cd        : chr [1:346] "CX" "" "CM0" "88" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_clin_stage_cd    : chr [1:346] "99" "" "1A" "88" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_seer_sum_stage   : int [1:346] NA NA NA 1 1 NA NA NA NA 9 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_tx_pre_path_stage    : chr [1:346] "" "No" "No" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_gen_sum_stage_2      : chr [1:346] "" "" "" "Unstaged" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ naaccr_tnm_edition_num  : chr [1:346] "5" "" "8" "88" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_dmets_yn             : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets1         : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets2         : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets3         : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets4         : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets5         : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets6         : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets7         : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets8         : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets9         : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_first_dmets10        : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_crc_td               : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_crc_crm              : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_crc_peri_inv         : chr [1:346] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_n_regimens           : int [1:346] 0 0 1 0 0 1 1 1 2 2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int        : int [1:346] 5556 1092 79 17 5088 9426 318 10822 1742 5085 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int_mos    : num [1:346] 182.763 35.921 2.599 0.559 167.368 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_lastalive_int_yrs    : num [1:346] 15.2115 2.9897 0.2163 0.0465 13.9302 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int            : int [1:346] 5567 1109 NA 111 NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int_mos        : num [1:346] 183.12 36.48 NA 3.65 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_death_int_yrs        : num [1:346] 15.242 3.036 NA 0.304 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_days           : int [1:346] 5567 1109 79 111 5088 9426 318 10822 1742 5085 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_mos            : num [1:346] 183.12 36.48 2.6 3.65 167.37 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_dx_yrs            : num [1:346] 15.242 3.036 0.216 0.304 13.93 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ os_status_dx            : int [1:346] 1 1 0 1 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_cohort              : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_i_adv_status        : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_adv_days       : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_adv_mos        : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_adv_yrs        : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_m_adv_status        : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_m_adv_days       : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_m_adv_mos        : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_m_adv_yrs        : logi [1:346] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version         : num [1:346] 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ ca_drugs          :'data.frame':	5453 obs. of  102 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                          : chr [1:5453] "CRC" "CRC" "CRC" "CRC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id                       : chr [1:5453] "GENIE-DFCI-001858" "GENIE-DFCI-111044" "GENIE-DFCI-008343" "GENIE-MSK-P-0025715" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution                     : chr [1:5453] "DFCI" "DFCI" "DFCI" "MSK" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ regimen_number                  : int [1:5453] 1 1 1 1 2 1 1 1 2 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_seq                          : int [1:5453] 1 1 1 0 1 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ redcap_ca_index                 : chr [1:5453] "No" "No" "No" "Yes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_num                       : int [1:5453] 3 3 3 3 3 4 4 3 2 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_admin                     : chr [1:5453] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_inst                      : chr [1:5453] "At external institution only" "At external institution only" "At the internal/native institution only" "At the internal/native institution only" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_firstinst                 : chr [1:5453] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_ct_yn                     : chr [1:5453] "No" "No" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_dc_ynu                    : chr [1:5453] "Yes" "Yes" "Yes" "Yes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ regimen_drugs                   : chr [1:5453] "Cyclophosphamide, Dactinomycin, Vincristine Sulfate" "Cyclophosphamide, Doxorubicin Hydrochloride, Vincristine Sulfate" "Doxorubicin Hydrochloride, Methotrexate, Vincristine Sulfate" "Fluorouracil, Leucovorin Calcium, Oxaliplatin" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_1                    : chr [1:5453] "Vincristine Sulfate(Kyocristine,Leurocristine Sulfate,Leurocristine sulfate,Oncovin,Vincasar,Vincosid,Vincrex)" "Vincristine Sulfate(Kyocristine,Leurocristine Sulfate,Leurocristine sulfate,Oncovin,Vincasar,Vincosid,Vincrex)" "Doxorubicin Hydrochloride(ADM,Adriacin,Adriamycin,Adriamycin Hydrochloride,Adriamycin RDF,Adriamycine,Adriblast"| __truncated__ "Oxaliplatin(1OHP,Ai Heng,DACPLAT,Dacotin,ELOXATIN,Eloxatine,JM83)" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_2                    : chr [1:5453] "Dactinomycin(Actinomycin A IV,Actinomycin C1,Actinomycin D,Actinomycin I1,Actinomycin IV,Actinomycin X 1,Cosmeg"| __truncated__ "Doxorubicin Hydrochloride(ADM,Adriacin,Adriamycin,Adriamycin Hydrochloride,Adriamycin RDF,Adriamycine,Adriblast"| __truncated__ "Vincristine Sulfate(Kyocristine,Leurocristine Sulfate,Leurocristine sulfate,Oncovin,Vincasar,Vincosid,Vincrex)" "Leucovorin Calcium(Calfolex,Calinat,Cehafolin,Citofolin,Citrec,Citrovorum Factor,Cromatonbic Folinico,Dalisol,D"| __truncated__ ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_3                    : chr [1:5453] "Cyclophosphamide(CTX,CYCLOcell,Carloxan,Clafen,Claphene,Cyclophosphamide,Cyclophosphan,Cyclophosphane,Cyclostin"| __truncated__ "Cyclophosphamide(CTX,CYCLOcell,Carloxan,Clafen,Claphene,Cyclophosphamide,Cyclophosphan,Cyclophosphane,Cyclostin"| __truncated__ "Methotrexate(Abitrexate,AlphaMethopterin,Amethopterin,Brimexate,CL14377,Emtexate,Farmitrexat,Fauldexato,Folex,L"| __truncated__ "Fluorouracil(5FU,5Fluorouracil,5Fluracil,AccuSite,Adrucil,Carac,Fluouracil,Flurablastin,Fluracedyl,Fluracil,Flu"| __truncated__ ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_4                    : chr [1:5453] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_5                    : chr [1:5453] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_1                : chr [1:5453] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_2                : logi [1:5453] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_3                : logi [1:5453] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_4                : chr [1:5453] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_drug_oth_5                : logi [1:5453] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_1             : int [1:5453] 917 5258 5435 6752 7785 8264 8814 8824 8862 8962 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_2             : int [1:5453] 917 5258 5435 6752 7785 8264 8814 8824 8862 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_3             : int [1:5453] 917 5258 5435 6752 7785 8264 8814 8824 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_4             : int [1:5453] NA NA NA NA NA 8264 8814 NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_startdt_int_5             : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_1               : int [1:5453] 1099 5624 6136 6822 7785 8434 9010 8906 9155 9141 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_2               : int [1:5453] 1099 5624 6136 6822 7785 8812 9010 8906 9155 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_3               : int [1:5453] 1099 5624 6136 6822 7785 8812 9010 8906 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_4               : int [1:5453] NA NA NA NA NA 8812 9010 NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_enddt_int_5               : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_1              : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_2              : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_3              : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_4              : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drugs_lastdt_int_5              : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_1             : int [1:5453] 9 14 30 39 2541 14 11 48 612 76 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_2             : int [1:5453] 9 14 30 39 2541 14 11 48 612 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_3             : int [1:5453] 9 14 30 39 2541 14 11 48 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_4             : int [1:5453] NA NA NA NA NA 14 11 NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_5             : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_1         : num [1:5453] 0.296 0.461 0.987 1.283 83.586 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_2         : num [1:5453] 0.296 0.461 0.987 1.283 83.586 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_3         : num [1:5453] 0.296 0.461 0.987 1.283 83.586 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_4         : num [1:5453] NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_drug_start_int_mos_5         : num [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_1            : int [1:5453] 183 367 702 71 1 171 197 83 294 180 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_2            : int [1:5453] 183 367 702 71 1 549 197 83 294 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_3            : int [1:5453] 183 367 702 71 1 549 197 83 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_4            : int [1:5453] NA NA NA NA NA 549 197 NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_int_5            : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_1 : int [1:5453] 183 367 702 71 1 171 197 83 294 180 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_2 : int [1:5453] 183 367 702 71 1 549 197 83 294 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_3 : int [1:5453] 183 367 702 71 1 549 197 83 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_4 : int [1:5453] NA NA NA NA NA 549 197 NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ drug_start_end_or_lastadm_int_5 : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_reg_start_int                : int [1:5453] 9 14 30 39 2541 14 11 48 612 76 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_reg_start_int_mos            : num [1:5453] 0.296 0.461 0.987 1.283 83.586 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_reg_start_int_yrs            : num [1:5453] 0.0246 0.0383 0.0821 0.1068 6.9569 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_any_int           : int [1:5453] 183 367 702 71 1 171 197 83 294 180 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_any_int_mos       : num [1:5453] 6.0197 12.0724 23.0921 2.3355 0.0329 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_any_int_yrs       : num [1:5453] 0.50103 1.00479 1.92197 0.19439 0.00274 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_any_int: int [1:5453] 183 367 702 71 1 171 197 83 294 180 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_any_mos: num [1:5453] 6.0197 12.0724 23.0921 2.3355 0.0329 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_any_yrs: num [1:5453] 0.50103 1.00479 1.92197 0.19439 0.00274 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_all_int           : int [1:5453] 183 367 702 71 1 549 197 83 294 180 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_all_int_mos       : num [1:5453] 6.0197 12.0724 23.0921 2.3355 0.0329 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_all_int_yrs       : num [1:5453] 0.50103 1.00479 1.92197 0.19439 0.00274 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_all_int: int [1:5453] 183 367 702 71 1 549 197 83 294 180 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_all_mos: num [1:5453] 6.0197 12.0724 23.0921 2.3355 0.0329 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ reg_start_end_or_lastadm_all_yrs: num [1:5453] 0.50103 1.00479 1.92197 0.19439 0.00274 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d1_days                   : int [1:5453] 13565 11951 13458 850 9424 1373 451 933 775 1651 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d2_days                   : int [1:5453] 13565 11951 13458 850 9424 1373 451 933 775 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d3_days                   : int [1:5453] 13565 11951 13458 850 9424 1373 451 933 NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d4_days                   : int [1:5453] NA NA NA NA NA 1373 451 NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d5_days                   : int [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d1_mos                    : num [1:5453] 446 393 443 28 310 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d2_mos                    : num [1:5453] 446 393 443 28 310 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d3_mos                    : num [1:5453] 446 393 443 28 310 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d4_mos                    : num [1:5453] NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d5_mos                    : num [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d1_yrs                    : num [1:5453] 37.14 32.72 36.85 2.33 25.8 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d2_yrs                    : num [1:5453] 37.14 32.72 36.85 2.33 25.8 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d3_yrs                    : num [1:5453] 37.14 32.72 36.85 2.33 25.8 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d4_yrs                    : num [1:5453] NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_d5_yrs                    : num [1:5453] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ os_status_g                     : int [1:5453] 1 0 0 0 0 1 0 1 1 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ os_status_d                     : int [1:5453] 1 0 0 0 0 1 0 1 1 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_g_days                    : int [1:5453] 13565 11951 13458 850 9424 1373 451 933 775 1651 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_g_mos                     : num [1:5453] 446 393 443 28 310 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_os_g_yrs                     : num [1:5453] 37.14 32.72 36.85 2.33 25.8 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_i_status_g                  : int [1:5453] NA NA NA NA NA 1 1 1 1 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_g_days                 : int [1:5453] NA NA NA NA NA 522 197 341 233 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_g_mos                  : num [1:5453] NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_i_g_yrs                  : num [1:5453] NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ pfs_m_status_g                  : int [1:5453] NA NA NA NA NA 1 1 1 1 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tt_pfs_m_g_days                 : int [1:5453] NA NA NA NA NA 835 198 348 237 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. .. [list output truncated]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ prissmm_imaging   :'data.frame':	26479 obs. of  43 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort            : chr [1:26479] "CRC" "CRC" "CRC" "CRC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id         : chr [1:26479] "GENIE-DFCI-111551" "GENIE-DFCI-111551" "GENIE-DFCI-111551" "GENIE-DFCI-111551" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution       : chr [1:26479] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ scan_number       : int [1:26479] 1 2 3 4 5 6 7 8 9 10 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scan_int    : int [1:26479] 17939 17995 17995 18128 18128 18296 18342 18358 18440 18440 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_scan_days      : int [1:26479] 28 84 84 217 217 385 431 447 529 529 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_scan_mos       : num [1:26479] 0.921 2.763 2.763 7.138 7.138 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_scan_yrs       : num [1:26479] 0.0767 0.23 0.23 0.5941 0.5941 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_ref_scan_int: int [1:26479] NA 17939 NA 17995 NA 18128 NA 18296 18358 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_ref_scan_days  : int [1:26479] NA 28 NA 84 NA 217 NA 385 447 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_ref_scan_mos   : num [1:26479] NA 0.921 NA 2.763 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_ref_scan_yrs   : num [1:26479] NA 0.0767 NA 0.23 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_inst_perf   : chr [1:26479] "Internal institution" "Internal institution" "Internal institution" "Internal institution" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_inst_inter  : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scan_type   : chr [1:26479] "CT" "CT" "CT" "CT" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ oth_scan          : logi [1:26479] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___1: chr [1:26479] "" "" "Brain/Head" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___2: chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___3: chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___4: chr [1:26479] "Chest" "Chest" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___5: chr [1:26479] "Abdomen" "Abdomen" "" "Abdomen" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___6: chr [1:26479] "Pelvis" "Pelvis" "" "Pelvis" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___7: chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_scansite___8: chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ scan_sites        : chr [1:26479] "Chest, Abdomen, Pelvis" "Chest, Abdomen, Pelvis" "Brain/Head" "Abdomen, Pelvis" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_ca          : chr [1:26479] "Yes, the Impression states or implies there is evidence of cancer" "Yes, the Impression states or implies there is evidence of cancer" "No, the Impression states or implies there is no evidence of cancer" "Yes, the Impression states or implies there is evidence of cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_overall     : chr [1:26479] "Progressing/Worsening/Enlarging" "Improving/Responding" "" "Improving/Responding" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite1     : chr [1:26479] "C77.2    Intra-abdominal lymph nodes" "C77.2    Intra-abdominal lymph nodes" "" "C77.2    Intra-abdominal lymph nodes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite2     : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite3     : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite4     : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite5     : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite6     : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite7     : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite8     : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite9     : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite10    : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite11    : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite12    : chr [1:26479] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite13    : logi [1:26479] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite14    : logi [1:26479] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ image_casite15    : logi [1:26479] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version   : num [1:26479] 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ prissmm_pathology :'data.frame':	7207 obs. of  341 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort           : chr [1:7207] "CRC" "CRC" "CRC" "CRC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id        : chr [1:7207] "GENIE-DFCI-111551" "GENIE-DFCI-111551" "GENIE-DFCI-111551" "GENIE-DFCI-111483" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution      : chr [1:7207] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_number : int [1:7207] 1 2 3 1 2 3 4 5 6 7 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_rep_number  : int [1:7207] 1 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_inst   : chr [1:7207] "Internal institution" "Internal institution" "Internal institution" "External institution" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_rep_inst    : chr [1:7207] "Internal institution" "Internal institution" "Internal institution" "External institution" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_int    : int [1:7207] 17911 18135 18366 19182 19217 19516 20189 20260 20403 20621 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_days: int [1:7207] 0 224 455 0 35 334 1007 1078 1221 1439 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_mos : num [1:7207] 0 7.37 14.97 0 1.15 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_yrs : num [1:7207] 0 0.6133 1.2457 0 0.0958 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_type   : chr [1:7207] "Surgical pathology" "Surgical pathology" "Surgical pathology" "Surgical pathology" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_num_spec    : int [1:7207] 3 4 1 1 2 2 2 2 3 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site1       : chr [1:7207] "C18.4    Transverse colon" "C77.2    Intra-abdominal lymph nodes" "C77.2    Intra-abdominal lymph nodes" "C18.9    Colon NOS" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site2       : chr [1:7207] "C48.1    Specified parts of peritoneum" "C77.2    Intra-abdominal lymph nodes" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site3       : chr [1:7207] "C77.2    Intra-abdominal lymph nodes" "C77.2    Intra-abdominal lymph nodes" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site4       : chr [1:7207] "" "C18.9    Colon NOS" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site5       : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site6       : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site7       : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site8       : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site9       : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site10      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site11      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site12      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site13      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site14      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site15      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site16      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site17      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site18      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site19      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site20      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site21      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site22      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site23      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site24      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site25      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site26      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site27      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site28      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site29      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site30      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site31      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site32      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site33      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site34      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site35      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site36      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site37      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site38      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site39      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site40      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site41      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site42      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site43      : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site44      : logi [1:7207] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_site45      : logi [1:7207] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu_any  : chr [1:7207] "No" "No" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_specimen_insitu: int [1:7207] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu1     : chr [1:7207] "No" "No" "No" "No" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu2     : chr [1:7207] "No" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu3     : chr [1:7207] "No" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu4     : chr [1:7207] "" "No" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu5     : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu6     : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu7     : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu8     : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu9     : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu10    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu11    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu12    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu13    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu14    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu15    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu16    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu17    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu18    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu19    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu20    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu21    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu22    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu23    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu24    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu25    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu26    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu27    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu28    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu29    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu30    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu31    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu32    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu33    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu34    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu35    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu36    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu37    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu38    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_insitu39    : chr [1:7207] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. .. [list output truncated]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ prissmm_md        :'data.frame':	28401 obs. of  12 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort          : chr [1:28401] "CRC" "CRC" "CRC" "CRC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id       : chr [1:28401] "GENIE-DFCI-000048" "GENIE-DFCI-000048" "GENIE-DFCI-000048" "GENIE-DFCI-000048" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution     : chr [1:28401] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_visit_number : int [1:28401] 1 2 3 4 5 6 7 8 1 2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_onc_visit_int: int [1:28401] 21737 21779 21807 21835 21863 21891 21947 22038 18698 18739 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_md_visit_days: int [1:28401] 23 65 93 121 149 177 233 324 177 218 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_md_visit_mos : num [1:28401] 0.757 2.138 3.059 3.98 4.901 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_md_visit_yrs : num [1:28401] 0.063 0.178 0.255 0.331 0.408 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_type_ca_cur  : chr [1:28401] "Colon Cancer" "Colon Cancer" "Colon Cancer" "Colon Cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_ca           : chr [1:28401] "No, the Impression/Plan states or implies there is no evidence of cancer" "No, the Impression/Plan states or implies there is no evidence of cancer" "No, the Impression/Plan states or implies there is no evidence of cancer" "No, the Impression/Plan states or implies there is no evidence of cancer" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ md_ca_status    : chr [1:28401] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version : num [1:28401] 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ tumor_marker      :'data.frame':	24715 obs. of  14 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort               : chr [1:24715] "CRC" "CRC" "CRC" "CRC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id            : chr [1:24715] "GENIE-DFCI-000048" "GENIE-DFCI-000048" "GENIE-DFCI-000048" "GENIE-DFCI-000147" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution          : chr [1:24715] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_number            : int [1:24715] 1 2 3 1 2 3 4 5 6 7 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_spec_collect_int  : int [1:24715] 21737 21945 22038 18985 19194 19194 19218 19321 19419 19440 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_tm_days           : int [1:24715] 23 231 324 464 673 673 697 800 898 919 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_tm_mos            : num [1:24715] 0.757 7.599 10.658 15.263 22.138 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_tm_yrs            : num [1:24715] 0.063 0.632 0.887 1.27 1.843 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_type              : chr [1:24715] "CEA" "CEA" "CEA" "CEA" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_num_result        : num [1:24715] 0 0 0.5 2.8 3.7 75 2.8 2.5 3.8 5.3 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_result_units      : chr [1:24715] "ng/mL" "ng/mL" "ng/mL" "ng/mL" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_normal_range_lower: num [1:24715] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ tm_normal_range_upper: num [1:24715] 2.5 2.5 2.5 2.5 2.5 35 2.5 2.5 2.5 2.5 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version      : num [1:24715] 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ cpt               :'data.frame':	1574 obs. of  26 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cohort                : chr [1:1574] "CRC" "CRC" "CRC" "CRC" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ record_id             : chr [1:1574] "GENIE-DFCI-000048" "GENIE-DFCI-000147" "GENIE-DFCI-000147" "GENIE-DFCI-000233" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ institution           : chr [1:1574] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_number            : int [1:1574] 1 1 2 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_report_int        : int [1:1574] 22108 18775 19488 20399 23956 21334 30077 17945 16478 30662 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ ca_seq                : int [1:1574] 0 0 0 2 1 0 1 0 0 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_number      : int [1:1574] 1 2 5 4 1 1 2 1 2 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_rep_number       : int [1:1574] 2 1 1 1 1 1 1 1 1 1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_genie_sample_id   : chr [1:1574] "GENIE-DFCI-000048-7508" "GENIE-DFCI-000147-6898" "GENIE-DFCI-000147-350443" "GENIE-DFCI-000233-7633" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_oncotree_code     : chr [1:1574] "COAD" "READ" "COADREAD" "COAD" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_sample_type       : chr [1:1574] "Primary tumor" "Primary tumor" "Metastasis site unspecified" "Metastasis site unspecified" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_seq_assay_id      : chr [1:1574] "DFCI-ONCOPANEL-2" "DFCI-ONCOPANEL-2" "DFCI-ONCOPANEL-3" "DFCI-ONCOPANEL-2" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_order_int         : int [1:1574] 22066 18731 19476 20356 23940 21283 30036 17918 16436 30634 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_seq_date          : int [1:1574] 2015 2015 2017 2015 2016 2015 2015 2016 2015 2016 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dob_cpt_report_days   : int [1:1574] 22108 18775 19488 20399 23956 21334 30077 17945 16478 30662 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ cpt_report_post_death : int [1:1574] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_cpt_rep_days       : int [1:1574] 394 254 967 3209 597 962 99 218 303 372 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_cpt_rep_mos        : num [1:1574] 12.96 8.36 31.81 105.56 19.64 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_cpt_rep_yrs        : num [1:1574] 1.079 0.695 2.648 8.786 1.634 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_cpt_days : int [1:1574] 0 133 934 1171 0 0 16 0 8 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_cpt_mos  : num [1:1574] 0 4.38 30.72 38.52 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dx_path_proc_cpt_yrs  : num [1:1574] 0 0.364 2.557 3.206 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_cpt_rep_days: int [1:1574] 394 121 33 2038 597 962 83 218 295 372 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_cpt_rep_mos : num [1:1574] 12.96 3.98 1.09 67.04 19.64 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ path_proc_cpt_rep_yrs : num [1:1574] 1.0787 0.3313 0.0903 5.5797 1.6345 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ release_version       : num [1:1574] 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 1.2 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ fusions           :'data.frame':	402 obs. of  9 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Hugo_Symbol         : chr [1:402] "PARK2" "RNF43" "BRAF" "TP53" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Entrez_Gene_Id      : int [1:402] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Center              : chr [1:402] "MSK" "MSK" "MSK" "MSK" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Sample_Barcode: chr [1:402] "GENIE-MSK-P-0002702-T01-IM3" "GENIE-MSK-P-0004140-T01-IM5" "GENIE-MSK-P-0004140-T01-IM5" "GENIE-MSK-P-0004140-T02-IM5" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Fusion              : chr [1:402] "PARK2-intragenic" "RNF43-intragenic" "BRAF-intragenic" "TP53-intragenic" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ DNA_support         : chr [1:402] "yes" "yes" "yes" "yes" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ RNA_support         : chr [1:402] "unknown" "unknown" "unknown" "unknown" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Method              : chr [1:402] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Frame               : chr [1:402] "unknown" "in frame" "in frame" "unknown" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ cna               :'data.frame':	930 obs. of  1503 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Hugo_Symbol                : chr [1:930] "ABCB11" "ABL1" "ABL2" "ACTB" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002643.6598     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008624.7126     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.003423.6929     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.007675.6942     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008889.6970     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.004251.6964     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008840.6985     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008622.7765     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.006859.7774     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008759.7792     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.007342.7811     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.000233.7633     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.009329.7667     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.004341.7674     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.007310.7753     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.009942.7682     : int [1:930] 0 1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001506.7563     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.000048.7508     : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.005963.10811    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008517.10877    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010689.10976    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010146.10979    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011320.10900    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010372.10884    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011405.10339    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011512.10189    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011676.10582    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011726.10621    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011291.10623    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010494.10654    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.000971.10958    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011485.10600    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.006715.10706    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.009381.11506    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.005455.10313    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010552.10110    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.003654.10980    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.000946.10743    : int [1:930] 0 1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.006237.10634    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.002499.10699    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011551.10700    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.007910.11299    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010547.11352    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010632.10572    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011006.11475    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011096.11378    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011660.11382    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001755.11394    : int [1:930] 0 1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011695.10716    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.003770.10725    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.001131.11269    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011358.11358    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011535.11255    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.008572.11075    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.000738.11390    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011391.10687    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011074.11054    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011182.11603    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010421.10688    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011599.11171    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011599.481757   : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011222.11823    : int [1:930] 0 1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011218.11641    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.007011.11659    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011010.10732    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.007183.11623    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010053.10234    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011396.11089    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011258.11748    : int [1:930] 0 -1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010242.11755    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011252.11577    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.000247.11259    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011260.11665    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.004815.11565    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.010862.11559    : int [1:930] 0 1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.013270.23816    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.017652.23659    : int [1:930] 0 1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.011888.23826    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.023633.23873    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.015256.23914    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.023427.23932    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.023723.23707    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.023755.23751    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.022876.23829    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.023634.23881    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.016815.23894    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.013710.23920    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037717.36216    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.036347.36256    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.035417.36270    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037792.36211    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.024579.36065    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.035767.36218    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.032144.36045    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037963.48454    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.035106.48568    : int [1:930] 0 1 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.031398.48578    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ GENIE.DFCI.037895.48432    : int [1:930] 0 0 NA NA NA 0 NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. .. [list output truncated]</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ..$ mutations_extended:'data.frame':	23257 obs. of  54 variables:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Hugo_Symbol                  : chr [1:23257] "TP53" "TP53" "MECOM" "SETBP1" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Entrez_Gene_Id               : int [1:23257] 7157 7157 2122 26040 4233 5295 5551 8289 2131 324 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Center                       : chr [1:23257] "DFCI" "DFCI" "DFCI" "DFCI" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ NCBI_Build                   : chr [1:23257] "GRCh37" "GRCh37" "GRCh37" "GRCh37" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Chromosome                   : chr [1:23257] "17" "17" "3" "18" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Start_Position               : int [1:23257] 7578406 7578263 168819995 42532220 116415069 67589138 72358811 27100192 119123072 112175212 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ End_Position                 : int [1:23257] 7578406 7578263 168819995 42532220 116415069 67589138 72358811 27100192 119123072 112175216 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Strand                       : chr [1:23257] "+" "+" "+" "+" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Variant_Classification       : chr [1:23257] "Missense_Mutation" "Nonsense_Mutation" "Missense_Mutation" "Missense_Mutation" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Variant_Type                 : chr [1:23257] "SNP" "SNP" "SNP" "SNP" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Reference_Allele             : chr [1:23257] "C" "G" "A" "G" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Seq_Allele1            : chr [1:23257] "C" "G" "A" "G" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Seq_Allele2            : chr [1:23257] "T" "A" "G" "A" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dbSNP_RS                     : chr [1:23257] "rs28934578" "rs397516435" "novel" "rs759901142" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ dbSNP_Val_Status             : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Sample_Barcode         : chr [1:23257] "GENIE-DFCI-000048-7508" "GENIE-DFCI-000048-7508" "GENIE-DFCI-000048-7508" "GENIE-DFCI-000048-7508" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Matched_Norm_Sample_Barcode  : chr [1:23257] "NORMAL" "NORMAL" "NORMAL" "NORMAL" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Match_Norm_Seq_Allele1       : int [1:23257] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Match_Norm_Seq_Allele2       : int [1:23257] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Validation_Allele1     : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Tumor_Validation_Allele2     : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Match_Norm_Validation_Allele1: logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Match_Norm_Validation_Allele2: logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Verification_Status          : chr [1:23257] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Validation_Status            : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Mutation_Status              : chr [1:23257] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Sequencing_Phase             : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Sequence_Source              : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Validation_Method            : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Score                        : chr [1:23257] "" "" "" "" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ BAM_File                     : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Sequencer                    : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ HGVSp_Short                  : chr [1:23257] "p.R175H" "p.R196*" "p.M687T" "p.R972Q" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ t_ref_count                  : int [1:23257] 548 408 135 538 202 120 192 376 266 114 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ t_alt_count                  : int [1:23257] 31 41 191 37 162 13 156 44 441 239 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_ref_count                  : int [1:23257] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_alt_count                  : int [1:23257] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Protein_position             : chr [1:23257] "175/393" "196/393" "687/1051" "972/1596" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ Codons                       : chr [1:23257] "cGc/cAc" "Cga/Tga" "aTg/aCg" "cGg/cAg" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ SWISSPROT                    : chr [1:23257] "P04637" "P04637" "Q03112" "Q9Y6X0" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ RefSeq                       : chr [1:23257] "NM_001126112.2,NM_001276761.1,NM_001276760.1,NM_000546.5,NM_001126118.1" "NM_001126112.2,NM_001276761.1,NM_001276760.1,NM_000546.5,NM_001126118.1" "NM_001105078.3" "NM_015559.2" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ t_depth                      : int [1:23257] 579 449 326 575 364 133 348 420 707 353 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ n_depth                      : int [1:23257] 0 0 0 0 0 0 0 0 0 0 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ FILTER                       : chr [1:23257] "PASS" "PASS" "PASS" "PASS" ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_AF                    : num [1:23257] 4.06e-06 4.06e-06 NA 8.12e-06 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_AFR_AF                : num [1:23257] NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_AMR_AF                : num [1:23257] NA NA NA 2.98e-05 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_ASJ_AF                : num [1:23257] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_EAS_AF                : num [1:23257] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_FIN_AF                : num [1:23257] NA NA NA NA NA NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_NFE_AF                : num [1:23257] 8.96e-06 8.95e-06 NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_OTH_AF                : num [1:23257] NA NA NA NA NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ gnomAD_SAS_AF                : num [1:23257] NA NA NA 3.25e-05 NA ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   .. ..$ mutationInCis_Flag           : logi [1:23257] NA NA NA NA NA NA ...</span>
<span class="r-in"><span><span class="co"># }</span></span></span>
<span class="r-in"><span></span></span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p></p><p>Developed by Statistical Core GENIE-BPC, Jessica Lavery, Michael A. Curry, Samantha Brown, Karissa Whiting, Axel Martin.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.5.</p>
</div>

    </footer></div>

  

  

  </body></html>

