<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="For patients with multiple associated next generation (NGS) sequencing
reports, select one unique NGS report per patient for the purpose of creating
an analytic dataset based on user-defined criterion, including OncoTree code,
primary vs. metastatic tumor sample, and earliest vs. most recent sample. If
multiple reports for a patient remain available after the user-defined
specifications, or if no specifications are provided, the panel with the
largest number of genes is selected by default. Sample optimization is
performed in the order that the arguments are specified in the function,
regardless of the arguments’ order provided by the user. Namely, the OncoTree
code is prioritized first, sample type is prioritized second and finally the
time is prioritized last. For patients with exactly one genomic sample, that
unique genomic sample will be returned regardless of whether it meets the
user-specified parameters. Running the select_unique_ngs() function will
ensure that the resulting dataset returned by merging the next generation
sequencing report data onto the cohort_ca_dx dataset returned by
create_analytic_cohort() will maintain the structure of cohort_ca_dx (either
one record per patient or one record per diagnosis). Currently, if multiple
diagnoses per patient are returned from create_analytic_cohort(), using
select_unique_ngs() will select a single NGS report per patient. In future
iterations, this will be updated so that one NGS report per diagnosis can be
selected."><title>Selecting corresponding unique next generation sequencing reports — select_unique_ngs • genieBPC</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Selecting corresponding unique next generation sequencing reports — select_unique_ngs"><meta property="og:description" content="For patients with multiple associated next generation (NGS) sequencing
reports, select one unique NGS report per patient for the purpose of creating
an analytic dataset based on user-defined criterion, including OncoTree code,
primary vs. metastatic tumor sample, and earliest vs. most recent sample. If
multiple reports for a patient remain available after the user-defined
specifications, or if no specifications are provided, the panel with the
largest number of genes is selected by default. Sample optimization is
performed in the order that the arguments are specified in the function,
regardless of the arguments’ order provided by the user. Namely, the OncoTree
code is prioritized first, sample type is prioritized second and finally the
time is prioritized last. For patients with exactly one genomic sample, that
unique genomic sample will be returned regardless of whether it meets the
user-specified parameters. Running the select_unique_ngs() function will
ensure that the resulting dataset returned by merging the next generation
sequencing report data onto the cohort_ca_dx dataset returned by
create_analytic_cohort() will maintain the structure of cohort_ca_dx (either
one record per patient or one record per diagnosis). Currently, if multiple
diagnoses per patient are returned from create_analytic_cohort(), using
select_unique_ngs() will select a single NGS report per patient. In future
iterations, this will be updated so that one NGS report per diagnosis can be
selected."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-primary"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">genieBPC</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.0.1.9002</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/clinical_data_structure_vignette.html">Tutorial: GENIE BPC Data Structure</a>
    <a class="dropdown-item" href="../articles/create_analytic_cohort_vignette.html">Tutorial: create_analytic_cohort</a>
    <a class="dropdown-item" href="../articles/drug_regimen_sunburst_vignette.html">Tutorial: drug_regimen_sunburst</a>
    <a class="dropdown-item" href="../articles/pull_data_synapse_vignette.html">Tutorial: pull_data_synapse</a>
    <a class="dropdown-item" href="../articles/select_unique_ngs_vignette.html">Tutorial: select_unique_ngs</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">Changelog</a>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/GENIE-BPC/genieBPC/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Selecting corresponding unique next generation sequencing reports</h1>
      <small class="dont-index">Source: <a href="https://github.com/GENIE-BPC/genieBPC/blob/HEAD/R/select_unique_ngs.R" class="external-link"><code>R/select_unique_ngs.R</code></a></small>
      <div class="d-none name"><code>select_unique_ngs.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>For patients with multiple associated next generation (NGS) sequencing
reports, select one unique NGS report per patient for the purpose of creating
an analytic dataset based on user-defined criterion, including OncoTree code,
primary vs. metastatic tumor sample, and earliest vs. most recent sample. If
multiple reports for a patient remain available after the user-defined
specifications, or if no specifications are provided, the panel with the
largest number of genes is selected by default. Sample optimization is
performed in the order that the arguments are specified in the function,
regardless of the arguments’ order provided by the user. Namely, the OncoTree
code is prioritized first, sample type is prioritized second and finally the
time is prioritized last. For patients with exactly one genomic sample, that
unique genomic sample will be returned regardless of whether it meets the
user-specified parameters. Running the select_unique_ngs() function will
ensure that the resulting dataset returned by merging the next generation
sequencing report data onto the cohort_ca_dx dataset returned by
create_analytic_cohort() will maintain the structure of cohort_ca_dx (either
one record per patient or one record per diagnosis). Currently, if multiple
diagnoses per patient are returned from create_analytic_cohort(), using
select_unique_ngs() will select a single NGS report per patient. In future
iterations, this will be updated so that one NGS report per diagnosis can be
selected.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">select_unique_ngs</span><span class="op">(</span></span>
<span>  <span class="va">data_cohort</span>,</span>
<span>  oncotree_code <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  sample_type <span class="op">=</span> <span class="cn">NULL</span>,</span>
<span>  min_max_time <span class="op">=</span> <span class="cn">NULL</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>data_cohort</dt>
<dd><p>output object of the create_analytic_cohort function.</p></dd>


<dt>oncotree_code</dt>
<dd><p>character vector specifying which sample
OncoTree codes to keep. See "cpt_oncotree_code" column
of data_cohort argument above to get options.</p></dd>


<dt>sample_type</dt>
<dd><p>character specifying which type of genomic sample
to prioritize, options are "Primary", "Local" and "Metastasis".
Default is to not select a NGS sample based on the sample type.</p></dd>


<dt>min_max_time</dt>
<dd><p>character specifying if the first or last genomic
sample recorded should be kept.
Options are "min" (first) and "max" (last).</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p>returns the 'cohort_ngs' object of the create_analytic_cohort
with unique genomic samples taken from each patients.</p>
    </div>
    <div class="section level2">
    <h2 id="details">Details<a class="anchor" aria-label="anchor" href="#details"></a></h2>
    <p>Note that the NGS dataset serves as the link between the clinical and
genomic data, where the NGS dataset includes one record per NGS report per
patient, including the NGS sample ID that is used to link to the genomic
data files. Merging data from the NGS report onto the analytic cohort
returned from create_analytic_cohort() therefore allows users to utilize all
clinical and genomic data available.</p>
<p>See the
<a href="https://genie-bpc.github.io/genieBPC/articles/select_unique_ngs_vignette.html">select_unique_ngs vignette</a>
for further documentation and examples.</p>
    </div>
    <div class="section level2">
    <h2 id="author">Author<a class="anchor" aria-label="anchor" href="#author"></a></h2>
    <p>Karissa Whiting</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co"># Example 1 ----------------------------------</span></span></span>
<span class="r-in"><span><span class="co"># Create a cohort of all patients with stage IV NSCLC of</span></span></span>
<span class="r-in"><span><span class="co"># histology adenocarcinoma</span></span></span>
<span class="r-in"><span><span class="va">nsclc_2_0</span> <span class="op">&lt;-</span> <span class="fu"><a href="pull_data_synapse.html">pull_data_synapse</a></span><span class="op">(</span><span class="st">"NSCLC"</span>, version <span class="op">=</span> <span class="st">"v2.0-public"</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">pt_char</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_index</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_dx_non_index</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">ca_drugs</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_imaging</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_pathology</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">prissmm_md</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cpt</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">mutations_extended</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">fusions</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">cna</span> has been imported for <span style="color: #0000BB;">"NSCLC_v2.0"</span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="va">ex1</span> <span class="op">&lt;-</span> <span class="fu"><a href="create_analytic_cohort.html">create_analytic_cohort</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  data_synapse <span class="op">=</span> <span class="va">nsclc_2_0</span><span class="op">$</span><span class="va">NSCLC_v2.0</span>,</span></span>
<span class="r-in"><span>  stage_dx <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Stage IV"</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>  histology <span class="op">=</span> <span class="st">"Adenocarcinoma"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># select unique next generation sequencing reports for those patients</span></span></span>
<span class="r-in"><span><span class="va">samples_data1</span> <span class="op">&lt;-</span> <span class="fu">select_unique_ngs</span><span class="op">(</span></span></span>
<span class="r-in"><span>  data_cohort <span class="op">=</span> <span class="va">ex1</span><span class="op">$</span><span class="va">cohort_ngs</span>,</span></span>
<span class="r-in"><span>  sample_type <span class="op">=</span> <span class="st">"Primary"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">63</span> patients with &gt; 1 next generation sequencing reports were identified</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">34</span> of <span style="color: #00BB00;">63</span> had a unique NGS report selected based on given criteria.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #BBBB00;">!</span> <span style="color: #0000BB;">29</span> of <span style="color: #00BB00;">63</span> did not have a unique NGS report selected based on the selected criteria or by having the largest panel, so a NGS report was selected at random (be sure to set a seed for reproducbility!) See `attributes(&lt;your-results&gt;)$random_samples` to view these sample IDs.</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># Example 2 ----------------------------------</span></span></span>
<span class="r-in"><span><span class="co"># Create a cohort of all NSCLC patients who</span></span></span>
<span class="r-in"><span><span class="co"># received Cisplatin, Pemetrexed Disodium or Cisplatin,</span></span></span>
<span class="r-in"><span><span class="co"># Etoposide as their first drug regimen</span></span></span>
<span class="r-in"><span><span class="va">ex2</span> <span class="op">&lt;-</span> <span class="fu"><a href="create_analytic_cohort.html">create_analytic_cohort</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  data_synapse <span class="op">=</span> <span class="va">nsclc_2_0</span><span class="op">$</span><span class="va">NSCLC_v2.0</span>,</span></span>
<span class="r-in"><span>  regimen_drugs <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="st">"Cisplatin, Pemetrexed Disodium"</span>,</span></span>
<span class="r-in"><span>    <span class="st">"Cisplatin, Etoposide"</span></span></span>
<span class="r-in"><span>  <span class="op">)</span>,</span></span>
<span class="r-in"><span>  regimen_order <span class="op">=</span> <span class="fl">1</span>,</span></span>
<span class="r-in"><span>  regimen_order_type <span class="op">=</span> <span class="st">"within regimen"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="va">samples_data2</span> <span class="op">&lt;-</span> <span class="fu">select_unique_ngs</span><span class="op">(</span></span></span>
<span class="r-in"><span>  data_cohort <span class="op">=</span> <span class="va">ex2</span><span class="op">$</span><span class="va">cohort_ngs</span>,</span></span>
<span class="r-in"><span>  oncotree_code <span class="op">=</span> <span class="st">"NSCLCPD"</span>,</span></span>
<span class="r-in"><span>  sample_type <span class="op">=</span> <span class="st">"Metastasis"</span>,</span></span>
<span class="r-in"><span>  min_max_time <span class="op">=</span> <span class="st">"max"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">14</span> patients with &gt; 1 next generation sequencing reports were identified</span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p></p><p>Developed by Jessica A. Lavery, Michael A. Curry, Samantha Brown, Karissa Whiting, Hannah Fuchs, Axel Martin.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer></div>

  

  

  </body></html>

